\begin{savequote}[8cm]
In asexual organisms, before the descendants can aquire a combination of beneficial mutations, these must first have occurred in sucession, within the same lines of decent.
  \qauthor{--- Muller \cite{Soumillon2013Cellular}}
\end{savequote}

\chapter{\label{app:1-A}Supplementary Figures}

\minitoc



\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Zcwpw1_GTEX.png}
	\includegraphics[width=\textwidth]{figures/zcwpw1/zcwpw1_human_isoforms.png}
	\caption[\textit{Zcwpw1} Tissue and Isoform expression]{
		\textbf{(A)} \textit{Zcwpw1} is highly and specifically expressed in the testis. Data Source: GTEx Analysis Release V7 (dbGaP Accession phs000424.v7.p2).
		\textbf{(B)} \textit{Zcwpw1} isoform expression in human testits.
	}
	\label{fig:isoforms}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/TeloFISH_labelled.png}
	\caption[Telomeric Localisation]{
		ZCWPW1 localises at subtelomeric regions of chromosomes in pseudo-pachytene cells. Testis chromosome spreads from \textit{Zcwpw1\textsuperscript{-/-}} were immunostained for SYCP3 and ZCWPW1, and hybridised by FISH with a telomeric probe. Note that the ZCWPW1 foci do not always exactly co-localize with the telomeric signal, and generally lie more internally on the synaptonemal complex axis. Right panel: 3x zoom from left image showing the area in the dashed white rectangle.
	}
	\label{fig:telofish}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/tangled.png}
	\caption[Tangled Chromosome Phenotype]{
		Asynapsis and lack of XY body formation in \textit{Zcwpw1\textsuperscript{-/-}} mouse testis. Testis chromosome spreads from WT, \textit{Zcwpw1\textsuperscript{-/-}} and \textit{Prdm9\textsuperscript{-/-}} mice were immunostained with antibodies against SYCP3, γ-H2AX (phosphorylated form) and HORMAD2, and counterstained with DAPI. WT Pachytene cells show full synapsis of all autosomes and an XY body strongly labelled with γ-H2AX. In contrast, the XY body is absent in \textit{Zcwpw1\textsuperscript{-/-}} and \textit{Prdm9\textsuperscript{-/-}} pseudo-Pachytene cells, with no clear XY body formation. In the \textit{Prdm9\textsuperscript{-/-}} mutant, mispairing of homologues is evident by the formation of branched structures referred to as “tangled” chromosomes in the text.
	}
	\label{fig:tangled}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/RAD51.png}
	\caption[RAD51 Counts]{
		\textbf{(A)} RAD51 staining in \textit{Zcwpw1\textsuperscript{-/-}} mouse testis. Testis chromosome spreads from \textit{Zcwpw1\textsuperscript{+/+}} and \textit{Zcwpw1\textsuperscript{-/-}} mice were immunostained with antibodies against the synaptonemal complex protein SYCP3 and the recombinase RAD51, and counterstained with DAPI to visualise nuclei. Developmental stages are indicated at the top.
		\textbf{(B)} The number of RAD51 foci in cells from the various stages of prophase I were counted. p-values are from Welch’s two sided, two sample t-test. L: Leptotene, Z: Zygotene, P: Pachytene. n=2 mice per genotype (\textit{Zcwpw1\textsuperscript{-/-}} and WT), n=1 for \textit{Prdm9\textsuperscript{-/-}}.
	}
	\label{fig:Rad51}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Zcw_vs_CoTransfection_summary_plot_allchr.pdf}
	\caption[Zcwpw1 alone vs Co-transfection]{
		Enrichment of ZCWPW1 when co-transfected with PRDM9 is dependent on the ability of ZCWPW1 to bind in the absence of PRDM9. Enrichment was force called in 100bp windows across the genome. Data is conditioned on having input coverage of greater than 5 and enrichment>0.01 for both axes. Hexagons are coloured if at least 3 data points are present. Solid lines show density contours estimated by MASS::kde2d() in R.
	}
	\label{fig:Zcw_vs_cotransfection}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/transfection_efficiency.png}
	\caption[Transfection Efficiency]{
		Co-transfection of ZCWPW1 and PRDM9 in HEK293T cells. Immunofluorescence staining against the protein tags shows high expression levels of each protein and a reasonable proportion of co-expressing cells with merged overlapping signals (ranging from light green to yellow and light red depending on the expression ratio of the two proteins).
	}
	\label{fig:transfection_efficiency}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/HeatmapA.png}
	\includegraphics[width=\textwidth]{figures/zcwpw1/HeatmapB.png}
	\caption[Zcwpw1 Heatmaps]{
		\textbf{(A)} Profiles and heatmaps of reads at locations of chimp PRDM9. Heatmaps show log fold change of sample (as indicated in the title of each column, methods) vs input, for the top ¼ of peaks of chimp (c) PRDM9, for various samples, ordered by chimp PRDM9. Here showing that ZCWPW1 is found at sites of chimp PRDM9 peaks, when co-transfected with chimp PRDM9, but not at human (h) PRDM9 peaks.
		\textbf{(B)} Profiles and heatmaps of reads at locations of ZCWPW1 co-transfected with human PRDM9. Heatmaps show log fold change of sample (as indicated in the title of each column, Methods) vs input, for the top ¼ of peaks of ZCWPW1 when co-transfected with PRDM9, for various samples, ordered by first column. Here showing that H3K4me3, H3K36me3 and hPRDM9 are found at ZCWPW1 peaks when co-transfected with PRDM9.
	}
	\label{fig:Heatmaps}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/DMC1_ROC_PR.pdf}
	\caption[DMC1 prediction]{
		ZCWPW1 enrichment (with PRDM9 vs without) provides a better predictor of DMC1 sites than PRDM9 itself using a logistic regression (methods section \ref{sec:prediction}).
		\textbf{(A)} Receiver Operating Characteristic curve
		\textbf{(B)} Precision Recall Curve. ZCWPW1 enrichment (with PRDM9 vs without) refers to enrichment of ZCWPW1 cotransfected with PRDM9 relative to (using as input) ZCWPW1 transfected alone.
	}
	\label{fig:ROC}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Histone_correlation.pdf}
	\caption[Histone Correlation from peaks]{
		Fraction of ZCWP1 peaks (co-transfected with PRDM9 with input coverage of at least 5) that overlap either H3K4me3 or H4K36me3 for different bins of ZCWPW1 enrichment (100 equal sample size bins). Error bars show ±2 s.e. of the proportion. ``Randomised'' shows expected proportions when x-axis regions are randomly shifted within a range of $10^8$ bases (up to a maximum of the length of the chromosome).
	}
	\label{fig:histone_corr}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/H3Kme3_vs_Zcw_wP9.pdf}
	\caption[Histone Correlation from windows]{
		Enrichment from 100bp non-overlapping windows is binned into 100 equal sample size bins and mean enrichment of ZCWPW1 cotransfected with PRDM9 is plotted. Error bars show ±2 s.e. of the mean. Windows with evidence of PRDM9-independent H3K4me3 have been removed from the H3K4me3 plot. Additionally, x-axis regions were removed if input reads were <15 and y-axis regions if <5.
	}
	\label{fig:histone_corr2}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/H3K4strength_allchr.png}
	\caption[Relative H3K4me3 Strength]{
		Dependence of ZCWPW1 enrichment on H3K4me3. Subplot titles describe human PRDM9 transfection status vs ZCWPW1 cotransfection (with human PRDM9) status. Enrichment was force called in 100bp windows across the genome. Data for each colour group was downsampled to have the same number of data points per group. Hexagons are coloured if at least 5 data points fall within that bin. Solid lines show density contours estimated by MASS::kde2d() in R.
	}
	\label{fig:H3K4strength_denisty}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Hotspotsharing.pdf}
	\caption[Hotspot Sharing]{
		Fraction of B6 hotspot locations seen in \textit{Zcwpw1\textsuperscript{-/-}} DMC1 ChIP-seq at different p-values. Heat refers to DMC1 signal. Black rung bars across the top each represent a single peak. We observe almost all of the peaks seen in B6 wild type, especially for the hottest hotspots.
	}
	\label{fig:hotspotsharing}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/DMC1_positions.png}
	\caption[DMC1 positions]{
		DSBs in \textit{Zcwpw1\textsuperscript{-/-}} are positioned at WT locations within hotspots. DMC1 signal (left and center) is stratified by SPO11 (right). Stratification is into active hotspots (top 30\%) with >90\% of the SPO11 signal in the central 300bp (green), and <50\% central, and >90\% upstream of the PRDM9 binding motif (red) or <10\% upstream (black).
	}
	\label{fig:DMC1_positions}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/HOP2.pdf}
	\caption[HOP2 KO Phenocopying]{
		\textit{Hop2\textsuperscript{-/-}} and \textit{Zcwpw1\textsuperscript{-/-}} mouse knockout mutants show the same linear relationship of DMC1 ChIP-seq vs SPO11 (WT). DMC1 data from \textit{Hop2\textsuperscript{-/-}} mice is from GSM851661 \parencite{Khil2012Sensitive}.
	}
	\label{fig:HOP2}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/heatRatio.pdf}
	\caption[DMC1 ratio regression]{
		Regression of DMC1 ratio on H3K4me3. For each B6 mouse hotspot (Methods), we calculated the ratio of DMC1 fragment coverage (summed over the central 3.5kb region spanning DMC1 hotspot enrichment in \textit{Zcwpw1\textsuperscript{-/-}} and WT mice, after subtracting mean background over the ±2.5-5kb distal regions, and removing any hotspot with mean SPO11 coverage over the surrounding ±5kb outside the hotspot >1, to exclude nearby hotspots). We excluded weak hotspots whose estimated SPO11 heat (within 2.5kb) or DMC1 WT heats were in the bottom 33\% (because accurate ratio estimation is not possible for these hotspots). Circles: the force-called H3K4me3 signal strength, vs. the ratio, for each of 9,318 resulting hotspots. Blue line: mean H3K4me3 signal versus the ratio of mean DMC1 fragment coverages, after binning by H3K4me3 signal into 50 equal sample size groups of hotspot, within each bin. The points and blue line show a strong and essentially linear relationship.
	}
	\label{fig:heatRatio}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/proportions_plot_v3.pdf}
	\caption[ChIPseq peaks proportions]{
		Proportion of ZCWPW1 peaks overlapping various other marks, ordered by enrichment of ZCWPW1 binding over input. hPRDM9: human PRDM9; ut: untransfected. Subset categories are labelled according to the following example: “ZCWPW1 alone, Not hPRDM9, + Not utH3K4me3 + Not Alu” means ZCWPW1 binding peaks in cells transfected with ZCWPW1 alone that do not correspond to human PRDM9 binding peaks, or H3K4me3 peaks (ChIP-seq data in untransfected cells), or Alu repeats.
	}
	\label{fig:proportions}
\end{figure}


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/CpG_general.png}
	\caption[CpG counts for various repeats]{
		CpG count around ZCWPW1 peaks (+/-150bp, with input coverage>5) is positively associated with ZCWPW1 enrichment in both peaks overlapping Alus and peaks not overlapping Alus, but not at L1M1-3, L1MA or L1P repeats. Error bars show ±2 s.e. of the mean. 
	}
	\label{fig:CpG_general}
\end{figure}