\begin{savequote}[8cm]
\textit{Es müsse eine Art der Kerntheilung geben, durch welche die im Mutterkern enthaltenen Ahnenplasmen dergestalt auf die Tochterkerne vertheilt werden, dass jedem Tochterkern nur die halbe Zahl derselben zukomme.}

There must be a form of nuclear division in which the ancestral germ plasms contained in the nucleus are distributed to the daughter nuclei in such a way that each of them receives only half the number contained in the original nucleus.
  \qauthor{--- \cite{Weismann1887Ueber}}
\end{savequote}

\chapter{\label{ch:1-intro}Background}

\minitoc


\section{Aims}
Meiosis is a type of cell division in which the number of chromosomes is reduced by half to produce haploid gametes (sperm and egg cells)~\parencite{Ohkura2015Meiosis}. The primary aim of this research was to delineate the gene expression programme of spermatogenesis (during which meiosis occurs). Specifically, to identify which genes are expressed, in which cells, their degree of expression, and relative timing during spermatogenesis. With this atlas of expression data, we additionally aimed to infer the regulatory network controlling these expression patterns, as well as to predict protein function and in doing so identify targets for further characterisation. Finally, we aimed to carry out such characterisation for at least one of these targets.


\section{Motivations}

\subsection{Sexual Reproduction}
Reproduction is a fundamental property of all known life. Of the two empires of life Eukaryotes/Eukarya are able to reproduce sexually, and almost all do. The dominance of this trait has been estimated at roughly 99.9\% \parencite{White1978Modes}, with examples of exceptions being the Bdelloid rotifers - who lost the ability to reproduce sexually 60 Mya, and appear to be evolving by horizontal gene transfer \parencite{Debortoli2016Genetic}. This shared characteristic was likely present in the common ancestor of this empire and hence synapomorphic \parencite{Bernstein2013Evolutionary}. The sexual life cycle consists of an alternating halving of the genetic material/ploidy (by meiosis) to form haploid cells, followed by karyogamy (nuclear fusion). Meiosis (from Ancient Greek, meíōsis, “a lessening”) is a specialised form of cell division, in which genetic material is exchanged between the parental genomes (recombination). Recombination and independent assortment of homologous chromosomes generate genetic diversity and hence enable natural selection and evolution. Recombination also ensures balanced segregation at the reductional division which follows (no naturally occurring eukaryotes have a single chromosome - although synthetic yeast has been engineered - \cite{Shao2018Creating}).

The mechanisms of meiosis have important consequences for the evolution, fertility, and speciation of sexually reproducing organisms~\parencite{Davies2016Reengineering,Hassold2007Origin}. 


\subsection{Disease}
As the process of gamete generation, errors in meiosis can cause genetic disease in offspring. In some sense the most extreme version is complete infertility, affecting around 10\% of humans (with many being of unexplained cause) \parencite{Datta2016Prevalence, Hamada2011Unexplained}. Other errors can be compatible with fertilisation but not full gestation resulting in miscarriage (affecting around 13\% of pregnancies overall, but over 50\% for mothers over the age of 45 years - \cite{Magnus2019Role}). Others errors are compatible with birth but result in incurable disease such as some aneuploidies \parencite{Hassold2007Origin} and microdeletion syndromes caused by non-allelic homologous recombination \parencite{Myers2008common}.


A better understanding of the mechanisms of spermatogenesis and meiosis could aid in the prevention of such diseases. Additionally, determining the factors required for \emph{in vitro} spermatogenesis could enable the treatment of male infertility in addition to facilitating further research \parencite{Zhou2016Complete}.

While enabling the production of healthy offspring is one aim of reproductive medicine, \emph{preventing} fertilisation (usually but not always, temporarily) is also extremely important for the health of both mother and child \parencite{Cleland2012Contraception}. Hence the development of contraceptives is another key goal of reproductive science, and one for which a clearer delineation of testis specific genes could aid \parencite{Schultz2003multitude}.


\subsection{Curiosity}
As with many scientific endeavours curiosity is also a major driver of this research. The testis is a most unusual tissue especially at the transcriptional level. The testis has the highest number of tissue enriched genes out of the 32 tissues profiled in the Human Protein Atlas ($>$ five-fold higher mRNA levels in one particular tissue as compared to any other), with 1,057 enriched genes out of a total 2,489 across the 32 tissues~\parencite{Djureinovic2014human,Mele2015Human, Uhlen2015Tissuebased, Uhlen2016Transcriptomics, TheHumanProteinAtlas2019human}. This is over twice as many as the second highest tissue the cerebral cortex which is perhaps considered the most complex tissue functionally, and with which the testis shares a surprising similarity \parencite{Guo2005Transcriptomic, Djureinovic2014human, Uhlen2015Tissuebased}. More recent studies using single-cell sequencing have underscored the distinct transcriptional state of the testis (\cite{Han2018Mapping} and figure \ref{fig:MCA_PCA}). The testis is also one of a few immune privileged sites in the human body \parencite{Fijak2006testis}.


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/intro/MCA_PCA.pdf}
	\caption[Testis PCA (MCA)]{PCA of single cell RNAseq of 50 different tissues and cell types from over 400,000 cells from the mouse. Data from~\cite{Han2018Mapping}}
	\label{fig:MCA_PCA}
\end{figure}


 The previous two properties have lead to the use of testis expressed genes as targets for cancer immunotherapy (cancer testis antigens) as when re-expressed in cancers they provide a distinguishing biomarker \parencite{Whitehurst2014Cause}.

Chromatin undergoes massive remodelling during spermatogenesis ultimately resulting in most (90/99\% in human/mouse) of the histones being replaced by crosslinked protamines, leading to the cessation of transcription \parencite{Rathke2014Chromatin}. This necessitates long term mRNA storage ($>$5 days) before translation \parencite{Kleene2013Connecting}.

Sperm are a highly specialised cell, despite being a single cell and unable to transcribe new mRNA they must perform many of the functions of the whole organism including motility, sensing the environment (chemotaxis), and defence from attack (from the female immune system) \parencite{Kaupp2008Mechanisms, Thompson1992Leukocytic}. Indeed they are able to survive and even fertilise after more than three days in the female reproductive tract \parencite{Gould1984Assessment,Wilcox1995Timing}.

Meiosis itself of course is also unique, with paternal and maternal homologous chromosomes being blissfully unaware of each other in somatic cells, until pairing in meiosis in the germline.

%Incomplete cytokinesis


\subsection{Mapping Projects of Science}
Much of science can be characterised as the creation of maps. The discovery, description and cataloguing of various types of elements, which is often dismissed as `merely descriptive', is a fundamental bedrock of many sciences \parencite{Grimaldi2007Why}. Maps are often useful in two senses, they serve as a reference of what's already known, but they also arrange current knowledge in such a way as to provide a prediction of the unknown. For example Mendeleev's periodic table both catalogued the known elements but left gaps for new discoveries. Equally with the Standard Model in physics. Darwin and others made extensive catalogues of past and present diversity of life, eventually resulting in the concept of evolution by natural selection. Cataloguing the ratios of offspring phenotypes lead to Mendel's laws.

Our maps of protein function are woefully incomplete, with more than 3,000 (15\%) of human genes not even having a biological process annotation (and a similar \% in the more experimentally amenable organisms \textit{S. cerevisiae} and \textit{S. pombe}) \parencite{Wood2019Hidden}. Even genes with at least one annotation or paper are often incompletely or even incorrectly characterised. Significant bias exists against initiating research into a gene of unknown function \parencite{Edwards2011Too, Stoeger2018Largescale, Haynes2018Gene}. This can be partially attributed to risk averse incentives in scientific funding and evaluation. In addition, for many proteins of unknown function there are limited tools available such as antibodies, making even simple characterisation difficult (for example immunofluorescence or immuno-precipitation based assays).
















\begin{figure}[H]
	\centering
\end{figure}






%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Measuring Gene Expression}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Expression is the process in which information contained in the genome is 
There are a number of different technologies available to quantify the amount of mRNA each providing different information.

\subsection{In Situ Hybridization}
In Situ Hybridisation uses a probe (either RNA or DNA) which is incubated with fixed cells and binds to a target mRNA by complementary binding. 

Radioactive labelling was low resolution, expensive, unstable, and dangerous. but this was overcome by biotinylated probes couples with secondary detection \parencite{Singer1982Actin} and later directly flourescent probes \parencite{Kislauskis1993Isoformspecific}.

This technique has been extended to measure individual mRNA molecules by using multiple probes per target RNA (single molecule smFISH) \parencite{Raj2008Imaging, Femino1998Visualization}. In addition to multiplexed detection to enable many (up to X) RNA targets to be measured at once \parencite{}.


\subsection{RT-qPCR}
Reverse transcription quantitative real-time PCR measures PCR products during the amplification by using fluorescent detectors \parencite{Gibson1996novel, Heid1996Real, Chiang1996Use}. This technique is able to detect a single molecule, however any spatial information is lost.

Due to the exponential amplification, small differences in reverse transcription, primer efficiency, or initial starting amounts and contamination can lead to inaccurate quantification. Therefore, efficiency and validated reference genes must be established by separate experiments \parencite{Bustin2009MIQE}.

\subsection{Northern Blot}
The Northern blot (RNA extraction, separation by gel electrophoresis, followed by transfer by blotting onto a membrane and finally detection by complementary hybridisation with a radioactive/fluorescent probe)  is one of the earliest methods for measuring mRNA levels \parencite{Alwine1977Method}. Northern blotting still remains a useful technique when information about the relative sizes of transcripts is required.

\subsection{Microarrays}
\section{Gametogenesis}
Gametogenesis is the process by which primordial germ cells develop into either spermatozoa in the male (spermatogenesis) or ovum in the female (oogenesis). Whilst equally important to spermatogenesis, oogenesis is studied far less (1,099 vs 359 articles indexed in PubMed 2018). This is likely partially due to difficulties in studying oogenesis compared to spermatogenesis. In oogenesis prophase I up to diplotene occurs in the ovary of the fetus before birth at which point meiosis is arrested (``Dictyate''), making experiments more challenging.

\subsection{Testis Anatomy}
Whilst the fundamental features of spermatogenesis and oogenesis are the same, spermatogenesis is organised into a specific structure. The testis is composed of seminiferous tubules surrounded by somatic accessory cells such as Leydig cells which produce testosterone. Germ cells are located within the tubule supported by a somatic cell type named Sertoli cells. The germ stem cells are located at the basal surface next to the tubule wall, and more differentiated germ cells are located progressively inwards towards the lumen/apical surface of the tubule (figure~\ref{fig:histology}).

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/intro/histology.png}
	\caption[Testis Anatomy]{Cross section of a seminiferous tubule in testis with the main cell types labelled, reproduced from~\cite{Junqueira2005Basic}}
	\label{fig:histology}
\end{figure}

Within a given region of tubule, entry into spermatogenesis is coordinated such that each developing germ cell belongs to a cohort of the same stage. The rate of entry is precisely timed (every 8.6 days in mice) and shorter than the total developmental time (35 days in mice, >70 in humans) such that in any given tubule section different generations of germ cells occur in layers of particular cell type combinations (figure \ref{fig:cycle}, \cite{Oakberg1956Duration,Clermont1969Duration, Heller1969Human}). This is called the cycle of the seminiferous epithelium and it's possible to stage each combination of cells using periodic acid‐fuchsin sulfurous acid staining (14 combinations in rat, and 12 in mouse) \parencite{Leblond1952Spermiogenesis, Leblond1952Definition, Oakberg1956description}. The cycle starts at a different time laterally along the tubule (in a so called ``wave''), ensuring continuous production of sperm, although the very first wave is synchronous - a property sometimes used to purify or enrich for specific stages.

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/intro/cycle.png}
	\caption[Seminiferous Cycle]{Association of different germ cells during the seminiferous epithelial cycle~\cite[Reproduced from ][]{Monesi1978Chapter}}
	\label{fig:cycle}
\end{figure}











\begin{figure}[H]
	\centering
\end{figure}











\section{Prior Work}

A number of previous studies have measured spermatogenic gene expression. Some studies such as GTEx (Genotype-Tissue Expression), HPA (Human Proteome Atlas), and FANTOM (Functional Annotation Of Mammalian genome) consortia have transcriptionally profiled many whole tissues in humans using RNAseq~\parencite{Mele2015Human,Uhlen2015Tissuebased,Uhlen2016Transcriptomics}. This data enables the identification of tissue specific genes and hence a preliminary list of proteins which could be involved in meiosis.  However, the testis is a complex amalgamation of different cell types and bulk tissue sequencing can generally not resolve the gene expression for different cell types instead resulting in a proportion weighted average. The relative inability to faithfully culture mammalian meiotic cells in vitro has also hampered the ability to generate higher resolution time-course data \parencite{Zhou2016Complete, Hikabe2016Reconstitution}.

Nonetheless a number of techniques have been developed to improve on whole tissue resolution. For example the initial wave of spermatogenesis is somewhat synchronous and so sampling during this time could in theory yield homogeneous cell types/stages. However it is likely there is some variation in synchronicity resulting in impure samples and uncertain classification~\parencite{Laiho2013Transcriptome,Ball2016Regulatory}. Some studies have combined this approach with an in silico deconvolution or machine learning algorithms~\parencite{Margolin2014Integrated,Li2013Identification}. One study utilised the greater synchrony of meiosis in fetal ovaries in combination with germ cell depleted mutant mice to partially delineate the meiotic prophase gene regulatory programme~\parencite{Soh2015Gene}. Other approaches include either size based centrifugal sorting~\parencite{Soumillon2013Cellular,Buard2009Distinct,Grabske1975Centrifugal} or FACS (Fluorescence Activated Cell Sorting) using a DNA stain~\parencite{daCruz2016Transcriptome}, both of which result in a limited number of (possibly quite heterogeneous) cell populations. Immunohistochemistry enables single cell resolution of protein expression but is low throughput~\parencite{Djureinovic2014human}.

%\parencite{Chalmel2007conserved, Schultz2003multitude, Almstrup2004Analysis,Johnston2008Stagespecific, Chalmel2014HighResolution}

% table to compare

%Other studies have transcriptionally profiled only testis tissue and have tried to target specific meiotic cells. 

Knowledge of the yeast gene regulatory programme of meiosis is relatively well known, however, the lack of sequence similarity and inability to culture mammalian meiotic cells has hampered efforts to translate this network to more complex eukaryotes~\parencite{Brar2011HighResolution,Mata2002transcriptional,Chu1998Transcriptional,Handel2010Genetics}.


\section{Statistical Methods}

Although the methods and workflow for analysing bulk RNAseq are now fairly mature and standardised, scRNAseq data has characteristics such as sparsity and high N which are poorly handled by bulk RNAseq tools. In addition scRNAseq data has opened up new analysis possibilities such as pseudotemporal ordering which is not possible with bulk data. This has motivated the development of many (538 as of 18th December 2019) new methods and analysis packages to meet the new demands and challenges of analysing scRNAseq data~\parencite{Zappia2018Exploring}. In fact, the total number of tools to analyse single cell RNAseq data is comparable to the total number of single cell RNAseq datasets (683) \parencite{Svensson2019curated}.

%The sparse nature and low signal-to-noise ratio of scRNAseq data is partly due to the low number of molecules being measured. For example many transcripts are present at \textless 10 copies per cell, and yet these transcripts could be important given protein to mRNA ratios commonly exceed 1,000~\cite{Lahtvee2017Absolute,Marguerat2012Quantitative}. In addition there is large natural variation in expression between comparable cells due to stochastic bursting gene expression kinetics~\cite{Raj2006Stochastic}. These challenges are compounded by inefficiencies and biases at each step of the library preparation procedure (cell lysis, mRNA capture, reverse-transcription, amplification) resulting in losses of 50 to 90\%~\cite{Islam2012Highly}.

There two major analysis modes for scRNAseq, each of which can be applied to either cells or genes. These two analysis ``modes'' are actually extremes of an analysis spectrum ranging from continuous to discrete depending on whether cells are considered to be separate groups or as part of a (developmental) trajectory. For the discrete case cells can be clustered into types, and differentially expressed marker genes can be inferred. Examples of these kinds of methods include k-means, hierarchical clustering, density clustering, or consensus clustering~\parencite{Zurauskiene2016pcaReduce,Kiselev2017SC3,Guo2015SINCERA,Satija2015Spatial}. For the continuous case cells can be ordered in a pseudo-timeline (or more generally arranged in lineage trees), and gene expression dynamics inferred. Pseudo-temporal ordering of cells is typically achieved by creating a minimum spanning tree and then projecting cells onto the shortest path to create a timeline~\parencite{Trapnell2014dynamics,Ji2016TSCAN}. Alternatively a principal curve / graph or seriation can be used~\parencite{Marco2014Bifurcation,Qiu2017Reversed}.

\subsection{Dimensionality Reduction}

These analysis methods, however, typically struggle with the extremely high dimensionality of scRNAseq (with each gene representing a dimension), sometimes referred to as the curse of dimensionality \parencite{Donoho2000HighDimensional}. Dimensionality reduction is therefore often performed on the gene by cell count matrix prior to application of these methods. The rationale behind this is that the data does not uniformly fill the high dimensional space, but actually lies on a manifold of lower dimension (the manifold hypothesis, \cite{Moon2018Manifold}), and so by projecting the cells onto this manifold the dimensionality can be greatly reduced while preserving much of the true structure in the data. This quasi-high dimensional data arises when there is a simpler underlying data generating process, for example a transcription factor may activate many different genes in the same cells, resulting in highly correlated gene expression and therefore in a sense ``redundant'' dimensions that provide little additional information. Equivalently, cells exist in a limited number of transcriptional states compared to all the possible states given all possible combinations of gene expression. Dimensionality reduction also enables visualisation of the data in 2 or 3D, greatly enhancing an analysts intuition when reasoning about a dataset.

For visualisation purposes non-linear dimensionality reduction techniques are most commonly used, especially t-SNE (t-distributed Stochastic Network Embedding), and more recently its derivative UMAP \parencite{McInnes2018UMAP,Becht2019Dimensionality}. But other methods are also used such as diffusion map based approaches (PHATE and URD \cite{Haghverdi2015Diffusion, Moon2019Visualizing, Farrell2018Singlecell}), force directed graph methods like SPRING \parencite{Weinreb2018SPRING, Wagner2018Singlecell}, locally linear embedding \parencite{Welch2016SLICER}, and Gaussian process latent variable models \parencite{Campbell2015Bayesian}.

However, the lack of interpretable parameters in these methods limits their use for inferring co-expressed gene sets. Linear factorisations are therefore often used (in addition to the non-linear methods), for example Principal Components Analysis (PCA) \parencite{Alter2000Singular, Green2018Comprehensive},  Independent Component Analysis (ICA) \parencite{Saunders2018Molecular}, Non-Negative Matrix Factorisation (NMF)~\cite{Brunet2004Metagenes, Kim2003Subsystem, Shao2017Robust, Zhu2017Detecting, Duren2018Integrative, Kotliar2018Identifying, Welch2019SingleCell} and Factor Analysis (\cite{Bernardo2003Bayesian, Buettner2015Computational}, \cite[reviewed in][]{Stein-OBrien2018Enter}).

% image to compare PCA, ICA, NNMF etc?

NNMF is often motivated by the positive nature of the original data, in addition to potentially increased interpretability for purely positively additive components which also naturally have a degree of sparsity in both the cell scores and gene loadings due to the positivity constraint. However, we note that latent factors which allow negative loadings have the potential to better capture transcriptional repression. 

These methods infer linear subspaces / hyperplanes in the high dimensional data enabling projection of the data and dimensionality reduction. Each component / factor represents a mode of variation in the data, which could arise due to either a biological or a technical process. The data is modelled as a weighted mixture of these components represented by the product of two lower rank matrices, just as the measured data represent the contribution of multiple underlying (latent) processes. The row vectors of the first matrix specify for each cell the mixing weights of each factor. The column vectors of the second matrix specify the contribution of each gene to the components. Put another way the column and row vectors of each matrix specify how active each cell and gene is for a given component. While these methods are typically less useful for visualisation (as the more restrictive linear structure limits the amount of structure that can be represented by 2 or 3 factors) the cell scores and gene loadings provide a direct way to interpret the low dimensional structure with the cell scores effectively soft clustering the cells and the gene loadings giving a set of co-expressed marker genes for each soft-cluster. In addition this linear reduction can still be used as input to the non-linear methods as a second visualisation step.

%\cite{Satija2015Spatial,Trapnell2014dynamics}

Some of these methods have been adapted specifically to account for the sparsity of scRNAseq datasets, the idea being that some technical effects caused ``dropouts'' that should be corrected or accounted for. ZIFA (Zero Inflated Factor Analysis) is one example, which accounts for the sparsity by including a zero-inflation component in the model \parencite{Pierson2015ZIFA}. However, it is likely this apparent inflation of zeros is actually an artefact of normalising count data and a negative binomial model fits well for tag based scRNAseq datasets \parencite{Vieth2017powsimR, Svensson2019Droplet, Townes2019Feature}.

Beyond matrix factorization, there are other frameworks with similar goals that have been applied successfully to single cell data. One set of methods are those based on neural networks, such as self-organizing maps \parencite{Loffler-Wirth2015oposSOM, Kim2015SingleCell} and auto-encoder neural networks (DCA \& scVI) \parencite{Eraslan2019Singlecell, Lopez2018Deep}. These methods (much like t-SNE) create a non-linear embedding of the high dimensional data, resulting in a lower dimensional set of scores for each cell. This approach does not, however, provide the equivalent to gene loadings and so an additional differential expression analysis on a hard clustering of the latent embeddings has to be performed in order to find genes associated with the latent dimensions. However, more recently a hybrid approach has been proposed sacrificing some of the flexibility to gain in interpretability \parencite{Svensson2019Interpretable}.


