\begin{savequote}[8cm]
\textit{Es m√ºsse eine Art der Kerntheilung geben, durch welche die im Mutterkern enthaltenen Ahnenplasmen dergestalt auf die Tochterkerne vertheilt werden, dass jedem Tochterkern nur die halbe Zahl derselben zukomme.}

There must be a form of nuclear division in which the ancestral germ plasms contained in the nucleus are distributed to the daughter nuclei in such a way that each of them receives only half the number contained in the original nucleus.
  \qauthor{--- \cite{Weismann1887}}
\end{savequote}

\chapter{\label{ch:1-intro}Introduction} 

\minitoc


\section{Mapping Projects of Science}

Much of science can be characterised as the creation of maps. The discovery, description and cataloguing of various types of elements, which is often dismissed as 'merely descriptive', is a fundamental bedrock of many sciences \cite{Grimaldi2007}. Maps are often useful in two senses, they server as a reference of what's already known, but also arrange current knowledge in such a way as to provide a prediction of the unknown. For example Mendeleev's periodic table both catalogued the known elements but left gaps for new discoveries. Equally with the Standard Model in physics. Darwin and others made extensive catalogues of past and present diversity of life, eventually resulting in the concept of evolution by natural selection. Cataloguing the ratios of offspring lead to Mendels laws.

The human genome project provided a single base pair map of our DNA, which not only accelerated research by providing a reference and enabling new bioinformatic analyses, but also uncovered many genes and conserved sequences of unknown function. Other atlases such as the Human Protein Atlas and GTEx which compiled gene expression data went some way to shed light on these genes by associating tissue specific genes with the function those tissues. Now the wave of single cell resolution expression atlases such as the Human Protein Atlas and Mouse Cell Atlas as well as other tissue specific maps including this one, provide even higher resolution clues for the function of unstudied genes.

\section{Meiotic Recombination}

\subsection{Significance and Motivation}
The mechanisms of meiosis have important consequences for the evolution, fertility, and speciation of sexually reproducing organisms~\cite{Davies2016Reengineering,Hassold2007Origin}. RNA sequencing (RNAseq) has enabled the probing of these mechanisms at a transcriptional level. However, our current knowledge of the meiotic gene expression programme is low resolution. This is partly due to heterogeneity of the cell population in testes which is averaged by bulk tissue RNAseq~\cite{YasuhiroFUJIWAR2014Differential} as well as the relative inability to culture mammalian meiotic cells in vitro~\cite{Zhou2016Complete}. The recent introduction of single cell RNAseq (scRNAseq)~\cite{Gawad2016Singlecell}, raises the possibility of identifying the gene expression profile associated with different stages of meiosis. This may enable the identification of the meiotic transcriptional network as well as aid in the discovery and understanding of meiotic mechanisms.

%The identification of genes expressed in meiosis and their relative timing of expression
%(which genes activate the transcription of which other genes)
%(which genes are expressed when and where)
%(for example if proteins acting in a complex are co-expressed)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Stages of Meiosis}
Meiosis is a type of cell division in which the number of chromosomes is reduced by half to produce haploid gametes (sperm and egg cells)~\cite{Ohkura2015Meiosis}. When meiosis proceeds correctly, recombination and independent assortment of homologous chromosomes generate genetic diversity and hence enable natural selection and evolution. However, errors in meiosis can cause infertility as well as aneuploidy~\cite{Handel2010Genetics,Hassold2007Origin}. Hence, understanding the molecular mechanisms of meiosis is a key goal of both the reproductive medicine and evolutionary research communities.

Some studies such as GTEx (Genotype-Tissue Expression), HPA (Human Proteome Atlas), and FANTOM (Functional Annotation Of Mammalian genome) consortia have transcriptionally profiled many whole tissues in humans~\cite{Mele2015Human,Uhlen2015Tissuebased,Uhlen2016Transcriptomics}. This data enables the identification of tissue specific genes and hence a preliminary list of proteins which could be involved in meiosis. Testis have the highest number of tissue enriched genes out of the 32 tissues profiled in the HPA (defined as at least five-fold higher mRNA levels in a particular tissue as compared to all other tissues), with 1,057 enriched genes compared to 2,489 in total over 32 tissues~\cite{TheHumanProteinAtlasHuman}. However, this could be due to generally pervasive transcription in germ cells due to permissive chromatin~\cite{Soumillon2013Cellular}. Furthermore, testes are a mixture of cells including spermatogonia, which give rise to primary and secondary spermatocytes, which themselves develop into early and late spermatids, Sertoli cells which engulf and nurse the spermatocytes, as well as Leydig (interstitial) cells which secrete androgen hormones~(figure~\ref{fig:histology}). Many of these cells are not meiotic and so the gene expression values from bulk tissue are an average of all these cells and so can not be used to fully delineate gene expression patterns in meiosis itself.

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/histology.png}
	\caption{Cross section of a seminiferous tubule in testis with the main cell types labelled, reproduced from~\cite{Junqueira2005Basic}}
	\label{fig:histology}
\end{figure}

Other studies have transcriptionally profiled only testis tissue and have tried to target specific meiotic cells. For example the initial wave of spermatogenesis is somewhat synchronous and so sampling during this time could in theory yield homogeneous cell types/stages. However it is likely there is some variation in synchronicity resulting in impure samples and uncertain classification~\cite{Laiho2013Transcriptome,Ball2016Regulatory}. One study utilised the greater synchrony of meiosis in fetal ovaries in combination with germ cell depleted mutant mice to delineate the meiotic prophase gene regulatory programme~\cite{Soh2015Gene}. Other approaches include either size based centrifugal sorting~\cite{Soumillon2013Cellular,Buard2009Distinct,Grabske1975Centrifugal} or FACS (Flourescence Activated Cell Sorting) using a DNA stain~\cite{daCruz2016Transcriptome}, both of which result in a limited number of (possibly quite heterogeneous) cell populations. Immunohistochemistry enables single cell resolution of protein expression but is low throughput~\cite{Djureinovic2014Human}. There is one study using single cell transcriptomics however they focus mainly on pre-meiotic foetal development~\cite{Li2017SingleCell}.

%Recent studies have combined this approach with an in silico deconvolution or machine learning algorithms~\cite{Margolin2014Integrated,Li2013Identification}.

Knowledge of the yeast gene regulatory programme of meiosis is relatively well known, however, the lack of sequence similarity and inability to culture mammalian meiotic cells has hampered efforts to translate this network to more complex eukaryotes~\cite{Brar2011HighResolution,Mata2002Transcriptional,Chu1998Transcriptional,Handel2010Genetics}.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Single Cell RNAseq Technologies}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Bulk RNAseq}


\subsection{Droplet Based Technologies}

\subsection{Extensions}

\section{Statistical Methods}

Although the methods and workflow for analysing bulk RNAseq are now fairly mature and standardised, scRNAseq data has characteristics such as sparsity and high N which are poorly handled by bulk RNAseq tools. In addition scRNAseq data has opened up new analysis possibilities such as pseudotemporal ordering which is not possible with bulk data. This has motivated the development of many (\textgreater 120) new methods and analysis packages to meet the new demands and challenges of analysing scRNAseq data~\cite{Zappia2017ScRNAtools}.

The sparse nature and low signal-to-noise ratio of scRNAseq data is partly due to the low number of molecules being measured. For example many transcripts are present at \textless 10 copies per cell, and yet these transcripts could be important given protein to mRNA ratios commonly exceed 1,000~\cite{Lahtvee2017Absolute,Marguerat2012Quantitative}. In addition there is large natural variation in expression between comparable cells due to stochastic bursting gene expression kinetics~\cite{Raj2006Stochastic}. These challenges are compounded by inefficiencies and biases at each step of the library preparation procedure (cell lysis, mRNA capture, reverse-transcription, amplification) resulting in losses of 50 to 90\%~\cite{Islam2012Highly}.

There two major analysis modes for scRNAseq: ordering of cells (in the case of differentiating cells), and clustering of cells into types (either to discover new cell types, or for downstream analysis on each type for example to define the transcriptional profile). However, before this is done dimensionality reduction is often performed on the (log transformed) gene by cell count matrix. Standard Principal Components Analysis (PCA) or Independent Component Analysis (ICA) are popular methods due to their speed and ease of interpretation~\cite{Satija2015Spatial,Trapnell2014Dynamics}. Other linear reductions such as Non-Negative Matrix Factorisation (NMF)~\cite{Shao2017Robust} and factor analysis~\cite{Buettner2015Computational} have also been used. Some of these methods have been adapted specifically for single cell analysis, for example ZIFA is a factor analysis which accounts for the sparsity of single cell data by including a zero-inflation component in the model (Zero Inflated Factor Analysis)~\cite{Pierson2015ZIFA}.

Nonlinear dimensionality reduction is also used, especially t-SNE (t-distributed Stochastic Network Embedding), but self organising maps, locally linear embedding, Gaussian process latent variable models, and diffusion maps have also been proposed~\cite{Kim2015SingleCell,Welch2016SLICER,Haghverdi2015Diffusion,Campbell2015Bayesian}.

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/pseudotime_methods.png}
	\caption{Comparison of pseudotime inference methods, reproduced from~\cite{Cannoodt2016Computational}}
	\label{fig:pseudotime_methods}
\end{figure}

After dimensionality reduction, downstream analysis such as clustering of cells is then achieved using methods such as k-means, hierarchical clustering, density clustering, or consensus clustering~\cite{Zurauskiene2016PcaReduce,Kiselev2017SC3,Guo2015SINCERA,Satija2015Spatial}. Pseudo-temporal ordering of cells is typically achieved by creating a minimum spanning tree and then projecting cells onto the shortest path to create a timeline~\cite{Trapnell2014Dynamics,Ji2016TSCAN} (figure~\ref{fig:pseudotime_methods}). Alternatively a principal curve or graph can be used~\cite{Marco2014Bifurcation,Qiu2017Reversed}.


% Similar aims (delineating lineage hierarchies and transcriptional networks) have been achieved using single cell RNA-seq in tissues other than testis, highlighting the feasibility of this study~\cite{Trapnell2014Dynamics,Treutlein2014Reconstructing,DurruthyDurruthy2014Reconstruction,Moignard2013Characterization,Stegle2015Computational}.


\subsection{Principal Components Analysis}

\subsection{Factor Analysis}

\subsection{Sparsity}

%Frequentist
%Bayesian