\begin{savequote}[8cm]
	The association of paternal and maternal chromosomes in pairs and their subsequent separation during the reducing division as indicated above may constitute the physical basis of the Mendelian law of heredity.
	\qauthor{--- \cite{Sutton1902morphology}}
\end{savequote}

\chapter{\label{ch:3-Zcw} Zcwpw1}

\minitoc

\section{Introduction}

Using single cell RNA-sequencing of mouse testis, we identified a set of genes co-expressed in (pre)leptotene cells which are highly enriched for genes involved in meiotic recombination (section \ref{sec:leptotene}). Zcwpw1, which ranks 3rd in this set after Prdm9 (2nd), is of unknown function but contains two recognised protein domains: CW and PWWP, shown to individually bind H3K4me3 and H3K36me3 respectively \parencite{He2010Complex, Rona2016PWWP}. PRDM9 deposits both H3K4me3 and H3K36me3 at sites it binds \parencite{Powers2016Meiotic}, and this methyltransferase activity is essential for its role in double strand break positioning \parencite{Diagouraga2018PRDM9}, suggesting these marks may be recognized by downstream protein(s). We therefore hypothesised that ZCWPW1 might co-localise to recombination hotspots, by binding the histone modifications deposited by PRDM9.

In humans, ZCWPW1 is specifically expressed in testis \parencite{Carithers2015novel, Uhlen2015Tissuebased} (figure \ref{fig:isoforms}). It is also one of 104 genes specific to meiotic prophase in murine fetal ovary \parencite{Soh2015Gene}, further suggesting a conserved meiotic function. This was confirmed by a recent study showing that ZCWPW1 is required for male fertility, with ZCWPW1 hypothesised to recruit the DSB machinery to hotspot sites \parencite{Li2019histone}. Here, we show that ZCWPW1 co-evolves with PRDM9, and is recruited to recombination hotspots by the combination of histone marks deposited by PRDM9. However ZCWPW1 is not required for the positioning of DSBs at PRDM9-bound sites, which occurs normally in ZCWPW1-null mice. Instead, ZCWPW1 is required for proper inter-homologue interactions: synapsis and the repair of DSBs. In ZCWPW1-null mice, DMC1 signals show strong perturbations, with signals at autosomal hotspots resembling those on the X-chromosome, which does not have a homologue. Thus, ZCWPW1 represents the first protein directly positioned by PRDM9 binding, but impacting homologous DSB repair.

\vspace{0.7cm}
\begin{framed}
\noindent All experimental work in this chapter was performed by Emmanuelle Bitoun, Daniela Moralli, and Gang Zhang. Anjali Hinch helped with SSDS processing, and section \ref{sec:coevolution} is by Simon Myers (apart from figure \ref{fig:domains}). Much of this chapter has been posted as a preprint \cite{Wells2019ZCWPW1}, on which I am co-first author.
\end{framed}

\section{ZCWPW1 co-evolves with PRDM9}
\label{sec:coevolution}
Based on publicly available databases, we identified likely ZCWPW1 orthologues in 167 species in total, aligned each to the human reference ZCWPW1 protein, and compared against a previous analysis of PRDM9 (method section \ref{sec:coevolution}, \cite{Baker2017Repeated}). The alignment reveals the regions containing the CW and PWWP domains to be the most conserved among species (Figure \ref{fig:domains}), while other parts of the protein appear to have been lost in some species. In addition, there is a region of moderate conservation downstream of the PWWP domain, not overlapping any known domain. Notably, an SYCP1 (SCP1) domain is annotated in the mouse protein only, which although only suggestive, is interesting given that SYCP1 physically connects homologous chromosomes in meiosis. In addition, protein threading suggests that the C terminal end of ZCWPW1 may contain a methyl-CpG binding domain (methods section \ref{sec:threading}, \cite{Lobley2009pGenTHREADER}).


Because of incomplete available DNA and protein sequences, we are almost certain to miss some species where ZCWPW1 is present, and similarly it is challenging to identify PRDM9 orthologues \parencite{Baker2017Repeated}. Despite this, we see extremely high overlap between PRDM9 and ZCWPW1 occurrence. All identified ZCWPW1 orthologues are in vertebrates, similar to identified PRDM9 orthologues \parencite{Baker2017Repeated}. 99 of 167 (149) species with ZCWPW1 (PRDM9) respectively have the other protein; an even higher fraction of species have a close relative with an identified orthologue: 131 species with ZCWPW1 are in a family possessing PRDM9. Only 8 species possess ZCWPW1 but are in an order with no identified species possessing PRDM9: three amphibians including Xenopus frogs, and five placental mammals. Given widespread conservation of PRDM9 among mammals other than canids \parencite{Baker2017Repeated}, these five might plausibly represent false negatives. Thus, ZCWPW1 appears to mainly occur in species, and even more clearly in groups of species, possessing PRDM9.


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Zcwpw1_domain_architechture_cons.pdf}
	\caption[Zcwpw1 Domains and Conservation]{
		Domain organisation (A) and evolutionary conservation (B) of ZCWPW1.
		\textbf{(A)} Protein domains in the human and mouse proteins (source: UniProt). Start and end positions of each domain are shown above and below the rectangles respectively. Prediction of SCP-1 domain from \parencite{Marchler-Bauer2004CDSearch} and of MBDs from \parencite{Lobley2009pGenTHREADER} (methods section \ref{sec:threading})
		\textbf{(B)} Conservation of human amino acids, normalised Jensen-Shanon divergence from \parencite{Capra2007Predicting, Johansson2010comparative} using multiple alignment of 167 orthologues (methods section \ref{sec:coevolution}).
	}
	\label{fig:domains}
\end{figure}


We identified ZCWPW1 orthologues across the full spectrum of vertebrates possessing PRDM9, including jawless, cartilaginous and bony fish, coelacanths, turtles, snakes and lizards, and mammals. Previous work \parencite{Baker2017Repeated} has identified at least 6 independent complete PRDM9 loss events; in birds and crocodiles; in three distinct groups of teleost fish (although these fish possess a PRDM9 “beta” orthologue with mutations in key catalytic amino acids within the SET domain); in canids; and tentatively, in amphibians. The first four losses appear to have been completely mirrored by corresponding ZCWPW1 losses, with no species previously studied by Baker et al. in these groups possessing a ZCWPW1 orthologue. For the latter two, we did find potential orthologues (e.g. in canids), but with mutations at positions in ZCWPW1 that are conserved among all PRDM9-SET-possessing species (methods section \ref{sec:coevolution}). Thus, there has been similar (co)evolution of presence/absence of ZCWPW1 and PRDM9.

We observe a greater number of species - 28 - possessing PRDM9 orthologues but having no relative closer than their class possessing an identified ZCWPW1 orthologue. All of these are bony fish. Strikingly, 24 of these 28 orthologues possess one or more mutations in their SET domain \iffalse supp table \fi, predicted to disrupt H3K4me3 and/or H3K36me3 deposition. In strong contrast, 94\% of PRDM9-containing species with non-mutant SET domains have a closer relative possessing ZCWPW1 (odds ratio = 90; indicative $p < 10^{-15}$ by FET, though we note these observations are not all independent). This implies that ZCWPW1 is most often lost, whenever mutations occur in the SET domain of PRDM9. Interestingly, PRDM9-containing species with non-mutant SET domains are near-identical to those also possessing SSXRD domains, and SSXRD has been reported as essential for PRDM9’s H3K4me3 methyltransferase activity at hotspots in vitro \parencite{Thibault-Sennett2018Interrogating}. However, other domains of PRDM9 (KRAB, and the zinc finger array) have been lost across multiple species, and it is hypothesized PRDM9 does not position recombination in these species \parencite{Baker2017Repeated}. Nonetheless, they mainly retain ZCWPW1 \iffalse supp table \fi. In conclusion, among vertebrates at least, it appears the set of species with ZCWPW1 is extremely similar to those species possessing PRDM9 with both an intact SET domain, and an SSXRD domain, with considerable evidence of co-evolution of gain/loss events for each protein. We note that this pattern is precisely what would be expected a priori, if ZCWPW1’s main function involves recognition of the histone modifications catalysed by PRDM9-SET during early meiosis, as predicted from functional considerations.

\section{ZCWPW1 localisation and  KO phenotypes} %\textit{Zcwpw1\textsuperscript{-/-}} 
To investigate the role of ZCWPW1 during meiosis in vivo, we produced an antibody against the full-length recombinant mouse protein \iffalse supp fig 2 \fi and studied the phenotype of a newly generated knockout (KO) mouse line for Zcwpw1, with a particular focus on fertility and meiotic recombination.

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/ZCW_staining_in_B6.png}
	\caption[Zcwpw1 Expression]{
		Expression of ZCWPW1 across meiosis prophase I in mouse testis. Nuclear spreads were immunostained with antibodies against ZCWPW1 (red) and the synaptonemal complex protein SYCP3 (green) which labels the chromosome axis, and counterstained with DAPI (blue) to visualise nuclei. Developmental stages indicated above. Yellow arrows show apparent ZCWPW1 foci at both ends of the synaptonemal complex. The dashed circle shows staining in the XY body. Imaging by Daniela Moralli.
	}
	\label{fig:ZCWPW1_expression}
\end{figure}

In testes from wild-type (WT) mice, we observe a dynamic localisation of ZCWPW1 protein (Figure \ref{fig:ZCWPW1_expression}), similar but non-identical to that reported in a recent study \parencite{Li2019histone}. ZCWPW1 shows a strong, punctate nuclear staining excluding the pericentromeric regions (clustered into chromocenters brightly stained with DAPI) in zygotene and early pachytene cells; we detected transcript expression in earlier pre-leptotene to leptotene cells \parencite{Jung2019Unified}. In pachytene cells, ZCWPW1 expression drops, with the protein now mainly localised in the XY body and as bright foci at the ends of the synaptonemal complex labelled by SYCP3, not previously observed using an antibody raised against a 174 base-pair (bp) C-terminal region of the protein \parencite{Li2019histone}. By diplotene, little expression is visible. Using FISH to label telomeric ends of chromosomes, we established that these discrete foci of ZCWPW1 are located at subtelomeric regions (figure \ref{fig:telofish}).

We next studied mice from a constitutive \textit{Zcwpw1} KO line (methods section \ref{sec:komice}), carrying a \textasciitilde1.5kb frameshift deletion encompassing exons 5 to 7 upstream of the CW domain, creating a premature stop codon resulting in the production and predicted degradation (by RNA-mediated decay) of a short 492bp (vs 1893bp for WT) transcript (figure \ref{fig:Fertility}A). Confirming this, ZCWPW1 expression was completely absent in testis protein extracts \iffalse supp fig \fi and chromosome spreads from \textit{Zcwpw1\textsuperscript{-/-}} mice, in zygotene cells (Figure \ref{fig:Fertility}B) and all other meiotic stages of prophase I where expression is detected in WT mice \iffalse supp fig \fi. Confirming recent findings in a different \textit{Zcwpw1\textsuperscript{-/-}} mouse \parencite{Li2019histone}, we observed no overt fertility phenotype in either sex in the heterozygous \textit{Zcwpw1\textsuperscript{+/-}} mice (data not shown). However \textit{Zcwpw1\textsuperscript{-/-}} male mice were sterile with complete azoospermia and reduced testis size (Figure \ref{fig:Fertility}B,D), while female mice retained fertility until around 7-8 months of age \iffalse supp table \fi, and otherwise both sexes develop normally. As in \cite{Li2019histone}, we observe widespread asynapsed chromosomes in male mice, marked by γH2AX and HORMAD2, persistent DMC1 foci marking unrepaired DSBs (Figure \ref{fig:DMC1_count}), no meiotic progression beyond (pseudo)pachytene, failure to form the sex body, and a complete absence of MLH1 foci marking recombination crossover sites (figure \ref{fig:tangled}). 

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/Fertility.png}
	\caption[Zcwpw1 KO Phenotypes]{
		Male \textit{Zcwpw1\textsuperscript{-/-}} mice show reduced testis size and asynapsis, similar to the \textit{Prdm9\textsuperscript{-/-}} mutant.
		\textbf{(A)} Schematic of the Zcwpw1 knockout mouse line. E: Exon. gRNA guideRNA. Sanger sequencing DNA chromatograms of wild-type (WT) and knockout (KO) mice encompassing the deletion are shown. The intron-exon organization is not to scale.
		\textbf{(B)} Immunofluorescence staining of testis nuclear spreads from \textit{Zcwpw1\textsuperscript{+/+}} and \textit{Zcwpw1\textsuperscript{-/-}} mice for ZCWPW1, the synaptonemal complex protein SYCP3 which labels the chromosome axis, or HORMAD2 which marks unsynapsed chromosomes.
		\textbf{(C)} Representative testes from 9-10 weeks old WT (+/+), Het (+/-) and Hom (-/-) Zcwpw1 KO mice are shown.
		\textbf{(D)} Paired testes weight was normalized to lean body weight. The p-value is from Welch’s two sided, two sample t-test. \iffalse supp table 3 \fi
		\textbf{(E)} Synapsis quantification in testis chromosome spreads immunostained with HORMAD2, as in (B). The percentage of pachytene cells with all autosomes fully synapsed is plotted by genotype; $n\ge50$ cells. Vertical lines are 95\% Wilson binomial confidence intervals. \iffalse raw data table 4 \fi
	}
	\label{fig:Fertility}
\end{figure}

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/DMC1_counts.png}
	\caption[DMC1 Counts]{
		Similar DMC1 count elevation in \textit{Zcwpw1\textsuperscript{-/-}} and \textit{Prdm9\textsuperscript{-/-}} mice, compared to wild-type.
		\textbf{(A)} Testis chromosome spreads from wild-type and \textit{Zcwpw1\textsuperscript{-/-}} mice were immunostained for DMC1 and SYCP3. Representative late pachytene cells are shown.
		\textbf{(B)} The number of DMC1 foci in cells from the various stages of prophase I were counted. p-values are from Welch’s two sided, two sample t-test. L: Leptotene, Z: Zygotene, P: Pachytene. n=3 mice per genotype (\textit{Zcwpw1\textsuperscript{-/-}} and WT), n=2 for \textit{Prdm9\textsuperscript{-/-}} \iffalse raw data \fi.
	}
	\label{fig:DMC1_count}
\end{figure}

Each of these properties resembles observations in the \textit{Prdm9\textsuperscript{-/-}} mutant, and so we compared to this mutant. In our \textit{Zcwpw1\textsuperscript{-/-}} male mice, >98\% of pachytene cells failed to properly synapse at least one pair of chromosomes (Figure \ref{fig:Fertility}B, E), similar to \textit{Prdm9\textsuperscript{-/-}} males. However, the nature of the synaptic defects observed differed\iffalse supp table 4\fi. In the \textit{Prdm9\textsuperscript{-/-}} mutant, 63.2\% of the pseudopachytene cells showed mispairing of non-homologous chromosomes in a typical branched structure\iffalse tangled in supp table 4\fi. In contrast, we only observed 24.8\% of \textit{Zcwpw1\textsuperscript{-/-}} pseudopachytene cells with this type of error, while the majority (74.5\%) of cells contained multiple bundles of HORMAD2-positive unsynapsed chromosomes, which resemble the XY body and may merge with the sex chromosomes\iffalse multibodies in supp table 5 \fi. These results imply that the \textit{Prdm9\textsuperscript{-/-}} mutant often mispairs chromosomes, while \textit{Zcwpw1\textsuperscript{-/-}} spermatocytes mainly fail to pair a subset of chromosomes at all. The expression levels and staining pattern of ZCWPW1 were not visibly altered in testis nuclear spreads (data not shown) and protein extracts from \textit{Prdm9\textsuperscript{-/-}} mice.

Comparing levels of DMC1 foci as a proxy for DSB repair, or a potential cause for the asynapsis in \textit{Zcwpw1\textsuperscript{-/-}} males, in both \textit{Zcwpw1\textsuperscript{-/-}} and \textit{Prdm9\textsuperscript{-/-}} mice foci count was significantly elevated from early zygotene onwards, indicating delayed repair of DSBs (Figure \ref{fig:DMC1_count}). However, we observed a wider spread for \textit{Zcwpw1\textsuperscript{-/-}} males. A similar increase was observed at pachytene stage in the levels of RAD51 (figure \ref{fig:Rad51}), another ssDNA-binding protein which functions in concert with DMC1, with the large majority of RAD51 and DMC1 foci co-localizing \parencite{Brown2015Small, Tarsounas1999RAD51}. In contrast, there was no difference in the levels of RPA2. In \textit{Zcwpw1\textsuperscript{-/-}} males, like in the \textit{Prdm9\textsuperscript{-/-}} mutant, DSBs form and recruit RPA2, RAD51 and DMC1 in similar numbers \parencite{Li2019histone}, but fail to repair efficiently, accompanied by asynapsis and meiotic arrest at pseudo-pachytene. Indeed, we observe late unrepaired DMC1 foci mainly on asynapsed chromosomes\iffalse supp fig 8 \fi.

We also performed analyses of which chromosomes fail to synapse. FISH analysis revealed that chromosomes 18 and 19 are most often among these asynapsed chromosomes, while grouping of chromosomes based on their size revealed some asynapsis of all chromosomes, but at higher levels for shorter chromosomes \iffalse supp fig 9 \fi. This is similar to results observed for infertile hybrid male mice \parencite{Bhattacharyya2013Mechanistic}, whose asynapsis is driven by lack of binding to the homologous chromosome by PRDM9 \parencite{Davies2016Reengineering}. 



\section{ZCWPW1 recruitment by PRDM9}
We previously studied the binding properties of human PRDM9 and established a genome-wide map in transfected human mitotic (HEK293T) cells by ChIP-seq \parencite{Altemose2017map}, observing binding to the majority of human meiotic recombination hotspots. Based on the presence of H3K4me3 and H3K36me3 recognition domains in ZCWPW1, we hypothesized that it would be recruited to PRDM9-bound genomic sites, where these marks are deposited upon binding in HEK293T cells \parencite{Altemose2017map}.


\begin{figure}[H]
	\centering
	\includegraphics[width=0.75\textwidth]{figures/zcwpw1/Figure_5.pdf}
	\caption[Zcwpw1 Allele Specificity]{
		\textbf{(A)} Enrichment of ZCWPW1 (with vs without PRDM9) at PRDM9 binding sites when co-transfected with PRDM9 with either Human or Chimp Zinc Finger. Q = quartile. Human PRDM9 sites are centered and stranded by the motif.
		\textbf{(B)} Profiles and heatmaps of reads around top 25\% of individual human (h) PRDM9 binding sites (rows). Heatmaps: log-fold change of target (indicated in column titles, methods section \ref{sec:heatmaps}) vs input, for various labeled target proteins, ordered by human PRDM9. ZCWPW1, H3K4me3 and H3K36me3 each become enriched at human PRDM9 sites, following (co-)transfection with human PRDM9.
		\textbf{(C)} ChIP-seq data and annotation in a genome plot illustrate the behaviour of ZCWPW1 and other factors. ChIP-seq tracks show fragment coverage. Tracks where PRDM9 is present are labelled ``w/ PRDM9'', and below corresponding tracks with out PRDM9. ZCWPW1 binds to Alus, CpG islands and other CpG-rich sequences even in the absence of PRDM9. On addition of PRDM9, ZCWPW1 becomes strongly enriched at PRDM9 binding locations (center left peak within DIO1).
	}
	\label{fig:Allele_Specific}
\end{figure}

To test this, we co-transfected HEK293T cells with full-length human HA-tagged ZCWPW1 and either no other protein, or full-length PRDM9 alleles carrying the human or chimpanzee ZF-array, as studied previously \parencite{Altemose2017map}, and then performed ChIP-seq against the ZCWPW1 HA tag. Confirming the recruitment hypothesis, in the presence of human PRDM9 and compared to cells lacking human PRDM9, ZCWPW1 shows a strong enrichment at human PRDM9 binding sites (figure \ref{fig:Allele_Specific}A, B). Notably, even without PRDM9 we observed sequence-specific binding of ZCWPW1 to many sites in the genome (Figure \ref{fig:Allele_Specific}C and \ref{fig:Zcw_vs_cotransfection}). However, upon transfection, 91\% (98\%) of the strongest 10,132 (3,016) peaks are PRDM9 binding sites (cf. 7\% expected overlap with randomised peaks), so PRDM9 is able to strongly reprogram ZCWPW1 binding, suggesting that non-PRDM9 peaks are bound more weakly. Because our transfection has <100\% efficiency (figure \ref{fig:transfection_efficiency}), some cells containing ZCWPW1 will not possess PRDM9, and so it is possible that an even greater fraction of ZCWPW1 is redirected to PRDM9 binding sites in cells where both proteins are present. 

Importantly, when co-transfecting with a modified version of PRDM9 in which the zinc finger array is replaced with that from chimp (which binds different locations in the genome \parencite{Altemose2017map}), we find that the enrichment at human binding sites disappears, but instead ZCWPW1 is enriched at chimp PRDM9 binding sites (Figure \ref{fig:Allele_Specific}A and \ref{fig:Heatmaps}). This perturbation experiment provides strong evidence that PRDM9 causes recruitment of ZCWPW1 (as opposed to for example independent recruitment of both proteins). Notably, the strength of binding of ZCWPW1 at these sites provides a better predictor of DSB formation (DMC1 sites) than does PRDM9 binding strength itself (figure \ref{fig:ROC}): although we show (see below) that ZCWPW1 is not directly involved in DSB positioning, this might suggest involvement of similar features to those it recognizes, in recruiting DSBs. 

We tested whether this recruitment might be mediated by the dual histone modifications H3K4me3 and H3K36me3. Consistent with this idea, ZCWPW1 binding is positively associated with levels of both H3K4me3 and H3K36me3 marks (figures \ref{fig:histone_corr} and \ref{fig:histone_corr2}). We examined transcription start sites, which possess H3K4me3 at high levels, but lack H3K36me3, observing some ZCWPW1 signal at these sites, but with a uniformly lower mean ZCWPW1 enrichment compared to those with evidence of PRDM9 binding (and hence both histone marks) (Figure \ref{fig:DualMark} and \ref{fig:H3K4strength_denisty}). Although it is not possible to measure H3K4me3 presence/absence in individual cells in our system, even the most weakly bound PRDM9 sites – which must therefore possess H3K4me3 in only a fraction of cells – show stronger ZCWPW1 enrichment than the strongest promoters, which are likely to have near 100\% H3K4me3 marking, and possess > 2-fold more H3K4me3 than even the strongest PRDM9 binding sites. We conclude that H3K4me3 alone endows only relatively weak binding, while PRDM9 is therefore able to recruit ZCWPW1 with a much greater efficiency than sites marked by H3K4me3 alone, suggesting that both histone modifications might aid efficient binding.


\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/H3K4strength_binned_allchr_copy.pdf}
	\caption[Dual mark recruitment]{
		PRDM9 (H3K4me3 and H3K36me3) is a stronger recruiter of ZCWPW1 than promoters (H3K4me3 only). H3K4me3 and ZCWPW1 were force called in 100bp windows. Each subset is defined as indicated in the legend with the additional constraint of requiring input fragment coverage >5 for ZCWPW1 and >15 for H3K4me3. For each subset H3K4me3 was split into 25 bins with equal number of data points. Horizontal bars: 2 standard errors of the mean. Vertical dotted bars: upper and lower quartiles. Grey ribbons show 2 standard errors for a Generalized Additive Model on log(mean H3K4me3 enrichment + 0.1). Dashed black horizontal line highlights that the mean enrichment of the highest bin for promoters is similar to that of the lowest bin for PRDM9 bound sites.
	}
	\label{fig:DualMark}
\end{figure}

\section{DSBs positioning and repair}
Previous work has shown that \textit{Prdm9\textsuperscript{-/-}} mice use a new set of DSB hotspots, localising at CpG islands and/or promoter regions \parencite{Brick2012Genetic}. Given that PRDM9 recruits ZCWPW1, one possible function of ZCWPW1 may be that, in turn, it recruits the DSB machinery and hence forms part of the causal chain in normal positioning of DSBs at PRDM9-specified hotspots. Alternatively, the \textit{Zcwpw1\textsuperscript{-/-}} mutant phenotypes we observe might reflect a more downstream role. To distinguish these hypotheses, we gathered data on DSB positioning and repair dynamics genome-wide, by carrying out single stranded DNA (ssDNA) sequencing (SSDS) by ChIP-seq against DMC1 \parencite{Khil2012Sensitive}.

In the \textit{Zcwpw1\textsuperscript{-/-}} mutant, we observed normal localisation of DMC1 at the expected B6 hotspots for this background, with no activity at \textit{Prdm9\textsuperscript{-/-}} hotspots (Figure \ref{fig:DMC1_SSDS}A). Moreover, individual wild-type hotspots appear always to be active in the \textit{Zcwpw1\textsuperscript{-/-}} mutant (figure \ref{fig:DMC1_SSDS}C): we only fail to see evidence of DMC1 signal in a subset of the weakest hotspots, where we are likely to lack statistical power (figure \ref{fig:hotspotsharing}). Thus DSBs occur in unchanged hotspot regions in \textit{Zcwpw1\textsuperscript{-/-}} males. To check if they occur at the same locations within hotspot regions, we leveraged data for SPO11-mapped DSB sites \parencite{Lange2016Landscape}. Specifically, we identified three sets of hotspots whose respective mapped breaks occur mainly upstream, central, or downstream of the PRDM9 binding site (Methods), and compared their DMC1 ChIP-Seq signal profiles, as well as their SPO11 signals (figure \ref{fig:DMC1_positions}). This revealed that in the \textit{Zcwpw1\textsuperscript{-/-}} males, as in the wild-type, DMC1 signals, though broader, mirror this break positioning. Therefore, ZCWPW1 is not required to specify hotspot locations, and neither does it strongly influence where DSBs occur within hotspots. 

One perhaps important difference between the \textit{Zcwpw1\textsuperscript{-/-}} and wild-type DMC1 profiles is that the latter signal grows slightly (up to \textasciitilde200bp) wider in hotspots with increasing PRDM9-induced H3K4me3 (Figure \ref{fig:DMC1_SSDS}B), while this effect is absent in the \textit{Zcwpw1\textsuperscript{-/-}} case. This might be explained by small differences in chromatin accessibility subtly impacting SPO11 locations, DNA end resection which generates the 3’ ssDNA tail to which DMC1 binds, or downstream processing differences altering DMC1 span in the mutant mice. 

\begin{figure}[H]
	\centering
	\includegraphics[width=0.78\textwidth]{figures/zcwpw1/DMC1_C.pdf}
	\caption[DMC1 SSDS ChIPseq]{
		\textbf{(A)} DSBs occur at normal hotspot locations in the \textit{Zcwpw1\textsuperscript{-/-}} mouse. Average coverage of reads from DMC1 SSDS ChIP-seq at previously mapped regions (methods section \ref{sec:chip}) in B6 WT (left) and \textit{Prdm9\textsuperscript{-/-}} (right) mice is shown, centered at the PRDM9 motif (left). DMC1 profiles from a WT mouse are shown in red, data from \parencite{Brick2012Genetic}.
		\textbf{(B)} Normalised DMC1 profile is plotted for WT and \textit{Zcwpw1\textsuperscript{-/-}}, stratified by H3K4me3. Greyed out lines show the alternative genotype for comparison.
		\textbf{(C)} Altered DMC1 signals in \textit{Zcwpw1\textsuperscript{-/-}} mice. Relationship between SPO11-oligos (measuring the number of DSBs) vs DMC1 (a measure of the number and persistence of DSBs) at each B6 hotspot for WT and \textit{Zcwpw1\textsuperscript{-/-}}. Green line is y=x for reference.
	}
	\label{fig:DMC1_SSDS}
\end{figure} 

Despite very similar DSB locations, we saw much greater, and systematic, differences in the strength of the DMC1 signal at individual hotspots. As previously \parencite{Davies2016Reengineering, Khil2012Sensitive}, we note that observed average DMC1 signal strength at a hotspot reflects the product of the frequency at which DSBs occur there, and the average length of time DMC1 remains bound to the ssDNA repair intermediate, with the latter reflecting DSB processing/repair time. In contrast, available SPO11 ChIP-Seq data \parencite{Lange2016Landscape} reflect mainly the former, the frequency of DSBs. We therefore compared DMC1 and SPO11 signal strength at each autosomal and X-chromosome hotspot, in WT and \textit{Zcwpw1\textsuperscript{-/-}} mice. In WT mice (as seen previously in other mice \parencite{Davies2016Reengineering} including, albeit somewhat more weakly, even sterile hybrids) the non-PAR X-chromosome shows a very strong elevation of DMC1 signal strength, reflecting the persistent DMC1 foci on this chromosome also visible using microscopy, at DSB sites that eventually repair using the sister chromatid. Moreover, in WT mice hotspot heat (with hotter hotspots having stronger H3K4me3 signal also) shows a sub-linear relationship between SPO11 and DMC1. This is thought to reflect a wider phenomenon of faster DSB repair occurring within those hotspots whose homologue is more strongly bound by PRDM9, i.e. those hotspots with a stronger H3K4me3 signal in the WT mouse \parencite{Davies2016Reengineering, Hinch2019Factors, Li2019highresolution}. The X-chromosome DMC1 elevation is in a sense an extreme case of slower repair of DSBs whose homologue is not PRDM9-bound, because no homologue exists in this case.

However, we see a striking departure in the \textit{Zcwpw1\textsuperscript{-/-}} mouse, where the X-chromosome behaves similarly to the autosomes in terms of DMC1 vs. SPO11 signal strength (figure \ref{fig:DMC1_SSDS}C). Moreover, we also see a simple linear relationship between DMC1 and SPO11 binding in this mouse. These data imply the relationship between DMC1 removal and PRDM9 binding is eliminated in this mouse, so PRDM9 (whether because ZCWPW1 directly aids the process, or indirectly) appears unable to ``assist'' in homology search in this mouse. Indeed, the data imply a widespread perturbation of DSB repair in this mouse, with autosomal DMC1 foci persisting as long as those on the X-chromosome. In the previously studied \textit{Hop2\textsuperscript{-/-}} mouse \parencite{Khil2012Sensitive, Petukhova2003Hop2, Smagulova2011Genomewide}, we identified a very similar pattern (figure \ref{fig:HOP2}): in this mouse, DMC1 is loaded onto ssDNA, but DSBs are not repaired at all, indicating that DSB repair pathways in \textit{Zcwpw1\textsuperscript{-/-}} mice are profoundly altered.

These results are consistent with, and extend, our microscopy observations (figure \ref{fig:DMC1_count}) that many DSBs persist in the \textit{Zcwpw1\textsuperscript{-/-}} mouse, and some may never repair. We are unable to say whether DSB repair involves the homologue or not in this mouse, but the partial synapsis we observe suggests some repair does likely occur.

To further understand DMC1 persistence at individual hotspots, we estimated the relative DMC1 heat of 9,318 autosomal hotspots in the \textit{Zcwpw1\textsuperscript{-/-}} mouse, compared to the WT mouse. Via linear regression, by far the best individual predictor of this ratio among H3K4me3, WT DMC1, and SPO11 was the level of H3K4me3 (r = 0.55, $p < 10^{-15}$; figure \ref{fig:heatRatio}, while r = 0.57 when using all 3 predictors together). The average DMC1 ratio changes around 9-fold from the most weakly to the most strongly H3K4me3-marked hotspots. Therefore, if DMC1 signal changes are indeed explained by slower DSB processing in the \textit{Zcwpw1\textsuperscript{-/-}} mouse at hotspots bound strongly by PRDM9, then this implies a very strong effect. Whatever the cause, the overall strong correlation implies perturbation of DMC1 behaviour is widespread, rather than impacting any small subset of hotspots. Moreover, it appears to be mainly controlled by local levels of PRDM9 histone modification, in keeping with the evidence that ZCWPW1 recognizes this mark at PRDM9-bound sites, on either the broken or (identical) homologous chromosome.

\section{ZCWPW1 binds CpG dinucleotides}
In addition to the strong PRDM9-dependent ZCWPW1 peaks described earlier, there are many locations in HEK293T cells at which ZCWPW1 binds, typically more weakly, and independently of PRDM9. Indeed we identified over 800,000 ZCWPW1 peaks. Surprisingly, a large proportion of these binding sites overlap Alu repeats (Figure \ref{fig:CpG}A and \ref{fig:proportions}) (of which there are 1.1 million in the human genome \parencite{Deininger2011Alu}). The weakest ZCWPW1 peaks overlap Alus most frequently, whilst the strongest peaks are depleted of Alus relative to chance overlap (Figure \ref{fig:CpG}A).

Because Alus are rich in CpGs (containing >23\% of human CpG dinucleotides, with deamination depleting CpGs in older Alus \parencite{Luo2014Dynamic}), we tested if binding correlates with CpG presence/absence. Indeed, the binding of ZCWPW1 to Alus depends on the presence of CpGs, which are mainly methylated \parencite{Gaysinskaya2018Transient}: Alus with no CpGs were bound at a rate lower than by chance, but almost all Alus containing 10-20 CpGs are bound (Figure \ref{fig:CpG}B). This dependence is not Alu-specific: even outside genomic repeats, 300-bp regions with zero CpGs have negligible probability of overlapping a ZCWPW1 binding site, while the overlap probability rises to >50\% for regions containing 10 or more CpGs, suggesting CpGs are essential for PRDM9-independent ZCWPW1 binding (figure \ref{fig:CpG}C,D and \ref{fig:CpG_general}). ZCWPW1 appears to have greater affinity for methylated CpG pairs, but retains some affinity even for non-methylated regions. This binding mode was unexpected a priori, and does not appear to be easily explained by patterns of H3K4me3/H3K36me3 in the genome, which do not concentrate as strongly in these regions.

\begin{figure}[H]
	\centering
	\includegraphics[width=\textwidth]{figures/zcwpw1/CpG_plot_alone.png}
	\caption[ZCWPW1 binds CpG rich sequences]{
		ZCWPW1 binds CpG-rich sequences such as Alu repeats.
		\textbf{(A)} Fraction of overlap of ZCWPW1 binding peaks with Alus, ordered by enrichment in ZCWPW1 binding. ZCWPW1 peaks are binned into 25 bins with equal number of data points, and means of both enrichment and overlap are plotted. Solid ribbon shows prediction from GAM logistic model. Dotted line shows overall means.
		\textbf{(B)} Rate of overlap of Alu repeats with ZCWPW1 peaks, for Alus with different numbers of CpG dinucleotides.
		\textbf{(C)} The probability of a 300bp window overlapping a ZCWPW1 peak increases with increasing CpG count in that window. Windows overlapping (by 10bp or more) Alus, other repeats, or CpG islands have been excluded. Methylation status of NA is for windows with 0 CpGs. Methylated regions are those with a methylated to unmethylated reads ratio of >0.75, and unmethylated <0.25 (methods section \ref{sec:cpgmeth})
		\textbf{(D)} Relative proportion of peaks with given numbers of CpGs $\pm150$bp from peak center. ZCWPW1 peaks are enriched in CpGs compared to random peak locations. ‘Meth’, methylated; ‘UnMeth’, unmethylated; NA, not applicable.
	}
	\label{fig:CpG}
\end{figure}


\section{Discussion}
Here we exploited previous work on single-cell transcript profiling of mouse testis, which has established spatio-temporal (and likely functional) components of co-expressed genes \parencite{Jung2019Unified}, to identify co-factors relevant to PRDM9 function in specification of recombination hotspots and DSB repair during meiosis. We show that the histone methylation reader ZCWPW1, which is both highly co-expressed and co-evolving with PRDM9, is recruited by PRDM9 to its binding sites, likely mediated through the recognition of the dual H3K4me3-H3K36me3 histone mark PRDM9 deposits. 

While we find co-evolution of ZCWPW1 with PRDM9, consistent with proteins functioning in the same pathway, this appears to depend on particular domains. ZCWPW1 shows weaker conservation among species outside the CW and PWWP domains, suggesting these are of high importance, and co-occurs with PRDM9 in species where the latter possesses a SET domain predicted to retain catalytic activity for H3K4me3/H3K36me3, which the domains of ZCWPW1 are predicted to recognise, strongly suggesting a partnership totally dependent on this recognition. This co-occurrence can persist, even in species where PRDM9 is not believed to position DSBs, suggesting the co-operation between these proteins does not completely depend on this aspect of PRDM9’s function. Interestingly, such a SET domain of PRDM9 mainly occurs when PRDM9 possesses a SSXRD domain, which in other proteins \parencite{Banito2018SS18SSX} recruits CXXC2 which binds to CpG islands – whether this connects to our observation of a potential CpG-binding domain in ZCWPW1, and CpG-binding behaviour in a human cell line, remains to be explored.

A recent study reported phenotyping of an independent \textit{Zcwpw1\textsuperscript{-/-}} mouse \parencite{Li2019histone}, and we observe similar patterns of infertility in males, and results consistent with gradual fertility decline in females, towards sterility from 8 months onwards. Why females deficient for ZCWPW1 are (at least initially) fertile is unknown, but resembles sexual dimorphism in loss-of-function mutants for other meiotic genes involved in homologous recombination, where females exhibit a milder fertility phenotype \parencite{Cahoon2019Leagues, Morelli2005Not, Zhang2019meiosisspecific}. However, we observe a distinct intracellular localisation of ZCWPW1 from that reported \parencite{Li2019histone}: while we also detect ZCWPW1 as a diffuse nuclear signal across the nucleus (though excluding the chromocenters) and localisation to the XY body in pachytene, we see an additional signal in mid-pachytene to diplotene cells at the ends of the synaptonemal complex. This might reflect the specificity of our polyclonal antibody, which is raised against the full-length mouse protein vs a monoclonal antibody raised against a C-terminal region, hence recognising a wider spectrum of ZCWPW1. Given that the \textit{Zcwpw1\textsuperscript{-/-}} mouse arrests in early pachytene, it is not possible to test later stages, so a caveat is that we cannot completely exclude off-target antibody recognition. If accurate, the late-stage localisation of ZCWPW1 at subtelomeric regions resembles other proteins (Speedy A, CDK2, TFR1, SUN1, TERB1/2, MAJIN, and KASH), which form a complex tethering the telomeres to the nuclear envelope to facilitate chromosome movement and pairing - essential for synapsis and fertility \parencite{Ashley2001Localization, Ding2007SUN1, Horn2013mammalian, Shibuya2014TRF1binding, Tu2017Speedy}. However, no evidence of impaired telomeric attachment was reported in the other \textit{Zcwpw1\textsuperscript{-/-}}, based on normal localisation of TRF1 \parencite{Li2019histone}.

We find, using a model system, that ZCWPW1 is strongly recruited to PRDM9-bound sites, in an allele-specific manner, and dependent on the level of H3K4me3 and H3K36me3 that PRDM9 deposits. Hence recruitment is most likely due to recognition of these marks by the CW and PWWP domains of ZCWPW1. Li and colleagues \parencite{Li2019histone} showed co-immunoprecipitation of H3K4me3 with ZCWPW1, but our results imply that PRDM9 recruits ZCWPW1 much more strongly than H3K4me3 alone, further suggesting that the combination of histone marks is key for ZCWPW1's recognition of PRDM9 sites. Indeed in cells with PRDM9, the strongest ZCWPW1 binding sites are almost all PRDM9-bound sites. To the best of our knowledge, this is the first protein identified that directly recognises PRDM9 binding sites.

Although it has been suggested that ZCWPW1 might recruit the DSB machinery to PRDM9 binding sites \parencite{Li2019histone, Spruce2019HELLS}, we find that DSB formation and positioning are completely unaffected by loss of ZCWPW1. However, the additional role PRDM9 plays besides DSB positioning, in enabling homologous chromosome pairing \parencite{Davies2016Reengineering}, malfunctions in \textit{Zcwpw1\textsuperscript{-/-}} mice. Unlike in wild-type animals, DSBs at sites bound more strongly by PRDM9 do not repair more quickly. Instead, in \textit{Zcwpw1\textsuperscript{-/-}} males, all DSBs behave similarly to those on the X-chromosome - which repairs DSBs very slowly, even in wild-type animals, and does not synapse. This implies an unexpected role for ZCWPW1: in aiding PRDM9 in its second function, synapsis. ZCWPW1 therefore offers a (thus far) unique protein, linking PRDM9 binding with its downstream functions \parencite{Davies2016Reengineering, Hinch2019Factors, Li2019highresolution} in aiding rapid DSB processing/repair at hotspots whose homologue is bound by PRDM9.

PRDM9 is thought to aid homology search/chromosome pairing by such binding to the homologous chromosome, and hybrids where hotspots are highly “asymmetric” (i.e. the homologues are not bound at the sites where DSBs occur, due to evolutionary hotspot erosion) show multiple features identical to \textit{Zcwpw1\textsuperscript{-/-}} mice. These include asynapsis (particularly of shorter chromosomes with fewer DSBs), persistent DMC1 foci (at the asymmetric hotspots), and complete sterility in males only. In \textit{Zcwpw1\textsuperscript{-/-}} mice, we infer that the level of PRDM9 binding to the homologue now makes no difference to DSB persistence, so these mice behave, in many ways, as if all hotspots are completely asymmetric. Thus, a parsimonious explanation of our findings is that ZCWPW1 functions by attaching to PRDM9-bound sites on the matching homologous chromosome, thus marking possible repair partner locations for DSB sites, to help pair homologues. This might e.g. lead to recruitment to the chromosomal axis (where DMC1 foci are found), for potential use as the homologous repair template in DSB repair, perhaps mediated by the SCP-1-like domain identified in some orthologous copies. Given we observe ZCWPW1 throughout the nucleus in early meiosis, such recruitment would likely be transient in nature. Alternatively, ZCWPW1’s part in mediating PRDM9’s downstream roles might also depend (or even, depend only) on it binding the chromosome on which DSBs occur.

Given that we still observe some synapsis, similar to \textit{Prdm9\textsuperscript{-/-}} mice, we suggest that a ``back-up'' mechanism, which is independent of PRDM9 binding level, is acting to attempt homology search/chromosome pairing in \textit{Zcwpw1\textsuperscript{-/-}} mice. In fact, some such mechanism is likely necessary, to repair DSBs happening within a subset of hotspots that are asymmetric due to naturally occurring variation (as seen in e.g. humans and heterozygous mice).

\iffalse
If our observed >9-fold changes in relative DMC1 levels for strongly vs. weakly bound hotspots purely reflect increased DMC1 persistence, it might be this mechanism is greatly slower than the PRDM9-dependent pathway for wild-type mice. In turn, this could explain many – or all – of the downstream phenotypes in these mice, as consequences of delayed homology search.
\fi


The simplest explanation of our observation that ZCWPW1 binds in a CpG dependent manner in vitro is that these sites might be directly bound by a methyl-CpG binding domain (MBD) putatively observed in the human, mouse, and coelacanth \textit{Zcwpw1}, among others. Alternatively, recruitment might be indirect, with ZCWPW1 forming a complex with an MBD containing protein, e.g. SETDB1 which is expressed in HEK293 cells, co-expressed with ZCWPW1 and PRDM9 \parencite{Jung2019Unified, Schultz2002SETDB1}, and interacts with several other identified PRDM9 partner proteins \parencite{Mulligan2008CDYL, Parvanov2017PRDM9}. Given the precisely regulated developmental changes in methylation status, ZCWPW1/PRDM9 protein abundance, and chromatin during meiosis \parencite{Gaysinskaya2018Transient, SeisenbergerStefanie2013Reprogramming} it is unclear whether ZCWPW1 will also weakly bind Alus in vivo. However, if CpG dinucleotides do retain weak in vivo ZCWPW1 binding affinity, and given that CpG methylation is associated with transposons, it is interesting to speculate whether ZCWPW1 might play a role in germline transposon recognition, silencing, or facilitate repair of DSBs occurring within transposable elements.

\textit{Zcwpw1} possesses a paralogue, \textit{Zcwpw2}, which is also co-expressed in testis with \textit{Prdm9} (although at a lower level), possesses both CW and PWWP domains, and has been shown to be able to recognise H3K4me3 at least \parencite{Liu2016Familywide}. Further studies are required to investigate what function if any \textit{Zcwpw2} has in meiosis. One interesting possibility is that \textit{Zcwpw2}, rather than \textit{Zcwpw1}, might act to help position DSBs – if so, its low abundance relative to \textit{Zcwpw1} would be analogous to the low number of DSBs per meiosis (\textasciitilde300) relative to the number of PRDM9 binding sites.

In this study we have shown that \textit{Zcwpw1} co-evolves with \textit{Prdm9}, binds to sites marked by PRDM9, and is required for the proper processing, removal of DMC1, and repair of DSBs. How exactly it mediates repair, if for example by recruitment of other proteins, remains to be determined. This study also further demonstrates that genes co-expressed with PRDM9 represent a rich source of undiscovered meiotic genes, with important functional implications for recombination, meiosis progression and ultimately fertility. ZCWPW1 impacts synapsis, downstream of DSB formation. Therefore the protein(s) responsible for recognising PRDM9-bound sites for DSB formation, and for aiding synapsis, are at least partly different, and we anticipate that future studies will likely uncover additional PRDM9-recruited proteins.

\section{Methods}
\subsection{Orthologue Alignment}
\label{sec:orthologues}
We identified ZCWPW1 orthologues across species using four data sources: first, we used BlastP (\url{blast.ncbi.nlm.nih.gov}), against the full-length human reference ZCWPW1 sequence (identifier NP\_060454.3, against nr\_v5 database) storing the top 1000 hits using the default parameters (set 1). Secondly, we downloaded the two sets of Ensembl identified ZCWPW1 orthologues (against ENSMUSG00000037108.13; ensembl.org, 105 orthologues), and identified NCBI ZCWPW1 protein orthologues (146 species, www.ncbi.nlm.nih.gov). Initial examination of identified orthologues revealed conservation among orthologues mainly of the ZW and PWWP domains; therefore, to find additional orthologues sequences we performed tBLASTn (against the nr/nt nucleotide collection 17th July 2019; top 1000 hits) to identify orthologues in the NCBI nucleotide database, to the partial sequence (amino acids 256-339 of the reference sequence NP\_060454.3), corresponding to the ZW and PWWP domains of ZCWPW1. Protein sequences were then aligned against full-length human ZCWPW1 (NP\_060454.3) using BLASTP2.9 \parencite{Altschul1997Gapped,Altschul2005Protein}. We obtained taxonomy information from the folder \url{https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/new_taxdump/}, for comparisons among species and between ZCWPW1 and PRDM9.

Previous work has identified highly conserved “KWR” and “PWWP” patterns among CW and PWWP domains, respectively \parencite{He2010Structural, Qin2014Structure}. We thus identified an initial set of “clear” ZCWPW1 orthologues, and then used these to identify further less precise matches. “Clear” orthologues are defined as proteins within set 1 containing perfect matches to both these sequences, and such that in the NCBI alignment, at least 39\% of each sequence aligns to the human protein, and conversely. This second step is required to avoid spurious matches to e.g. ZCWPW2, which overlaps 19\% of ZCWPW1, by requiring >2-fold this match length; although several ZCWPW2 copies are in our initial list, no gene annotated as most similar to ZCWPW2 attains the 39\% overlap. For proteins within the “clear” set, we identified the longest alignment for each given species, resulting in 136 species with a likely ZCWPW1 orthologue. This initial screen identified three groups of fish, placental mammals, marsupials, monotremes and reptiles as likely possessing ZCWPW1, similar to our final conclusions. Moreover, it defined amino acids 247-428 in the human reference sequence as being conserved in the alignment (at most 1 sequence not aligning), including the annotated ZCW and PWWP domains.

Using this initial set, we identified additional orthologous sequences and refined our results, by identifying conserved bases within ZCWPW1. Specifically, we divided the 136 species into major clades, as in \parencite{Baker2017Repeated}, and gave each sequence a weighting so that the overall weight for each clade was the same (so e.g. each Placental mammal sequence was downweighted as this clade was over-represented). Within the consistently aligned region 247-428, we calculated the overall weighted probability of each of the 20 amino acids, or a gap (adding $10^{-5}$ to exclude zero weights). This identified 31 completely conserved amino acids, and 78 amino acids whose entropy was below 1 (equivalent to 2 amino acids, having equal probability, so implying one amino acid present >50\% of the time). Finally, we defined ZCWPW1 orthologues as those sequences matching at least 90\% of those 31 perfectly conserved bases which were aligned, and aligning to at least 50\% of these bases (this last condition allows for inclusion of incomplete sequencing or protein assembly). We note that while this 90\% condition is arbitrary, by definition all annotated orthologues exceed this threshold, while thresholds below ~70-80\% are exceeded by orthologues of ZCWPW2 among other genes, offering some justification. Nonetheless, it is worth pointing out that our analysis does not rule out the existence of more poorly conserved ZCWPW1 copies in more distantly related species. This approach identified a final set of 167 genes, which we annotated as likely ZCWPW1 orthologues and used for the majority of results. For each ZCWPW1 orthologue, we also identified the taxonomic relationship of the closest species possessing such a PRDM9 copy \iffalse(Supplementary Table 1)\fi. We also reciprocally annotated each PRDM9 copy previously identified \parencite{Baker2017Repeated} according to the closest species also possessing ZCWPW1\iffalse(Supplementary Table 1)\fi.

Our analyses revealed a relationship between ZCWPW1 predicted functional domains and PRDM9 histone modifications. Therefore, we identified additional potential conserved sites by identifying perfectly conserved bases among those species possessing both ZCWPW1 and PRDM9 orthologues, and where three key SET domain catalytic amino acids within PRDM9 are intact, meaning PRDM9 is predicted to have normal histone modification activity \parencite{Baker2017Repeated}. This identified a slightly larger number of conserved amino acids (37). Only eight of these varied in any of the 167 species with a potential ZCWPW1 orthologue: 260, 404 and 411 (in two Xenopus frogs, with 411 also in white-headed capuchin), 19 and 22 (in three canines), 25 (in Ocelot gecko), 257 (in Anolis lizard and in Wombat), 325 (in elephantfish). While these changes might alter ZCWPW1 function, the true behaviour of ZCWPW1 is uncertain in these cases, though it is interesting that canids and frogs represent clades that all appear to have lost PRDM9, and possess multiple, clustered, amino acid changes. 

\subsection{Domain Search}
\label{sec:threading}
pDomThreader \parencite{Lobley2009pGenTHREADER}, was used via the PSIPRED server (\url{http://bioinf.cs.ucl.ac.uk/psipred/}) on the following uniProt amino acid sequences (Q9H0M4|ZCPW1\_HUMAN, Q6IR42|ZCPW1\_MOUSE, E2RFJ2|E2RFJ2\_CANLF, M3XJ39\_LATCH, A0A3S5ZP38\_BOVIN, M3WDY6\_FELCA, and G3ULT5|G3ULT5\_LOXAF) all of which identified a match to 1ub1A00 in the C terminal section after the PWWP domain, except in LOXAF in which this match was slightly below the p-value threshold of 0.001.

SCP1 prediction used NCBI conserved domain search server (\url{https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi?}) with the uniProt amino acid sequences Q9H0M4|ZCPW1\_HUMAN and Q6IR42|ZCPW1\_MOUSE.

\subsection{Mice and genotyping}
\label{sec:komice}
KO mice for \textit{Zcwpw1} (C57BL/6N-Zcwpw1\textsuperscript{em1(IMPC)Tcp}) were generated by the Toronto Centre for Phenogenomics (Canada). A 1485-bp deletion on chromosome 5 spanning exons 5 to 7 of \textit{Zcwpw1} (chr5:137799545-13780101029) was engineered by CRISPR/Cas9 using guide RNAs 5’-GACTGCACTCACGGCCATCT-3’. The frameshift deletion introduces a stop codon in Exon 8, leading to a predicted unstable short truncated 492bp transcript. Mice were genotyped at the \textit{Zcwpw1} locus using the following primers, and standard cycling conditions: KO allele-Forward, 5’-CACAGGCTCATGTATGTTTGTCTC-3’; KO allele-Reverse, 5’-CTGCTTCGTCCTCTTTCCTTATCTC-3’; WT allele-Forward, 5’-TGCCACCACACTTCATTTGT-3’; WT allele-Reverse CCTGTTTCCTTCCCAACTCA-3’. The deletion was verified by direct Sanger sequencing of the KO genotyping PCR product (Source Bioscience, UK), following purification with the QIAquick PCR Purification Kit (Qiagen). Sequence analysis was carried out using Chromas LITE (version 2.1.1). 

KO mice for \textit{Prdm9} were described previously \parencite{Hayashi2005histone} and obtained from the RIKEN BioResource Research Center in Japan (strain name B6.129P2-Prdm9\textsuperscript{tm1Ymat}, strain number RBRC05145). Mice were genotyped at the \textit{Prdm9} locus using the following primers, and standard cycling conditions: WT allele-Forward 5’-AGGAATCTTCCTTCCTTGCTGTCG-3’; WT allele-Reverse 5’-ATTTCCCTGTATCTTCTTCAGGACT-3’; KO allele-Reverse 5’-CGCCATTCAGGCTGCGCAACTGTT-3’.

All animal experiments received local ethical review approval from the University of Oxford Animal Welfare and Ethical Review Body (Clinical Medicine board) and were carried out in accordance with the UK Home Office Animals (Scientific Procedures) Act 1986.

\subsection{Fertility measurements}
\label{sec:fertility}
Fertility was assessed in mice ranging from 9 to 12 weeks of age, either by mating with WT littermates and recording the average litter size and frequency, or by measuring paired testes weight (normalized to lean body weight), sperm count (per paired epididymides) and chromosome synapsis rate (by immunostaining of pachytene spermatocytes) in males. Lean body weight was measured using the EchoMRI-100 Small Animal Body Composition Analyzer.

\subsection{Immunostaining of spermatocytes}
\label{sec:immunostaining}
Mouse testis chromosome spreads were prepared using surface spreading \parencite{Barchi2008ATM, Peters1997dryingdown} and immunostained as previously described \parencite{Davies2016Reengineering}. The following primary antibodies were used: custom ZCWPW1 rabbit antiserum (1:100), mouse anti-SYCP3 (Santa Cruz Biotechnology sc-74569, D-1) or biotinylated rabbit anti-SYCP3 (Novus NB300-232), rabbit anti-DMC1 (Santa Cruz Biotechnology sc-74569, D-1), rabbit anti-HORMAD2 (Santa Cruz Biotechnology sc-82192), mouse anti-RAD51 (Abcam ab88572), rabbit anti-RPA2 (Abcam ab10359), mouse (Millipore 05-636, clone JBW301) or chicken (Orbit orb195374, discontinued) anti-phospho γ-H2AX and Alexa Fluor 488-, 647- or 594-conjugated secondary antibodies against rabbit, mouse or chicken IgG (ThermoFisher Scientific), as well as avidin 647 (ThermoFisher Scientific) were used to detect the primary antibodies. Images were acquired using either a BX-51 upright wide-field microscope equipped with a JAI CVM4 B\&W fluorescence CCD camera and operated by the Leica Cytovision Genus software, or a Leica DM6B microscope for epifluorescence, equipped with a DFC 9000Gt B\&W fluorescence CCD camera, and operated via the Leica LASX software. Image analysis was carried out using Fiji (ImageJ-win64). The specificity of the ZCWPW1 signal was verified using the pre-immune serum from the same rabbit: no staining was visible at any stage of prophase I (data not shown).

\subsection{FISH}
\label{sec:fish}
Following immunostaining of spermatocytes, telomeres were labelled using the Telomere PNA FISH Kit/Cy3 (Agilent), following the manufacturer’s instructions, but without protease treatment. For the identification of chromosome 18, BAC RP24-144B10 (CHORI, BACPAC Resource Center) was labelled with the Abbot Molecular Nick Translation kit, and applied to the immunostained spermatocytes. The hybridisation and signal detection were carried out using standard techniques.
Plasmids

Constructs encoding full-length human (h) or chimp (c) PRDM9 with a C-terminal V5 or N-terminal YFP tag for expression in mammalian cells (pLENTI CMV/TO Puro DEST backbone vector, Addgene plasmid \# 17293; \cite{Campeau2009Versatile} were described previously \parencite{Altemose2017map}. Constructs encoding full-length human (h) and mouse (m) ZCWPW1 for expression in mammalian cells were purchased from Genescript (hZCWPW1 cDNA with a C-terminal HA tag in pCDNA3.1; clone ID OHu16813) and Origene (mZCWPW1 cDNA with a FLAG-Myc dual tag in pCMV6-Entry; clone ID MR209594), respectively. For expression in E. Coli, full-length mZCWPW1 cDNA was subcloned into the pET22b(+) vector (Novagen) with a C-terminal poly-His tag. 

\subsection{ZCWPW1 antibody production}
\label{sec:zcwAb}
pET22b-mZCWPW1-His construct was transformed into BL21 (DE3) E.Coli cells (Invitrogen). Bacterial cultures were grown to a density with O.D600\textasciitilde0.7, and expression of recombinant His-tagged ZCWPW1 protein was induced overnight at 20°C by addition of IPTG to 0.5mM. The protein was purified using Talon metal affinity resin, according to the manufacturer’s instructions (Takara). Further purification was carried out by size exclusion chromatography (Superdex HiLoad 200 16/60, GE Life Sciences). Western Blot validation was carried out as described previously \parencite{Altemose2017map}, using mouse anti-human ZCWPW1 (SIGMA SAB1409478) and mouse anti-polyhistidine (SIGMA H1029). The purified protein was used to immunize 2 rabbits (Eurogentec, Belgium), and the resulting immune antisera were tested against the recombinant antigen by ELISA (Eurogentec), and the endogenous protein in mouse testes from WT and KO mice (data not shown), alongside the preimmune sera. 

\subsection{Cell line, transfection, and IF staining}
Human embryonic kidney (HEK) 293T cells were cultured and transfected using Fugene HD as previously described \parencite{Altemose2017map}. High and comparable expression of the target proteins was verified by immunofluorescence staining of duplicate transfected cultures before proceeding to ChIP. PRDM9 expression was visualized either directly from live cells through YFP fluorescence, or by immunofluorescence staining of fixed cells using rabbit anti-V5 (Abcam ab9116, 1:500) as described previously \parencite{Altemose2017map}. ZCWPW1-HA was detected by immunostaining as above, using rabbit anti-HA (Abcam ab9110, 1:100). The fraction of cells co-expressing ZCWPW1-HA and PRDM9-V5 was determined by co-staining with both antibodies, as above (figure \ref{fig:transfection_efficiency}).

\subsection{Immunoprecipitation and Western Blot detection}
\label{sec:IP}
Cell and testes protein extracts were prepared in lysis buffer containing 50mM Tris-HCl pH8.0, 150mM NaCl, 1\% Triton X-100 and a cocktail of protease inhibitors (SIGMA), followed by gentle rotation at 4°C for 30 to 45 min, respectively. Cell debris were pelleted by centrifugation at 4°C for 20 min at 20,000g. Protein extracts were quantified using the BCA assay (Pierce) and equal quantities were incubated overnight at 4°C with 5µl of rabbit polyclonal ZCWPW1 antiserum. Immunocomplexes were pulled down with 50µl of sheep anti-rabbit Dynabeads M-280 (Life Technologies) for 2h at 4°C with gentle rotation. After 5 washes in lysis buffer they were eluted from the beads by boiling in Laemmli sample buffer for 5min, and resolved on either 8\% or 4-20\% Novex Tris-Glycine precast gels (Invitrogen). Proteins were transferred onto PVDF membranes and the proteins of interest were detected by western blotting following standard procedures. Blots were blocked for 1h at room temperature (RT) in PBS containing 0.2\% Tween-20 (PBS-T) and 5\% milk, and incubated for 1h at RT with rabbit anti-ZCWPW1 (1:1,000 dilution), washed 3 washes in PBS-Tween buffer and incubated for 1h at RT with HRP-conjugated sheep anti-rabbit IgG antibody (GE Healthcare). Protein signals were revealed using the ECL Prime western blotting detection reagent according to the manufacturer’s recommendations (GE Healthcare). The specificity of the ZCWPW1 signal observed in B6 testis extract was verified by immunoprecipitation using the pre-immune serum from the same rabbit: no signal was detected (data not shown).

\subsection{ChIP-seq}
\label{sec:chip}
DMC1 ChIP-seq data from WT B6 mouse testis were generated previously \parencite{Brick2012Genetic}. H3K4me3 ChIP-seq data was generated previously \parencite{Davies2016Reengineering}.

\subsubsection{ChIP}
ChIP against ZCWPW1-HA, (h/cPRDM9-V5) was carried out from transfected HEK293T cells as previously described \parencite{Davies2016Reengineering, Altemose2017map} were crosslinked for 10min in 1\% formaldehyde, the reaction was quenched for 5min by the addition of glycine to a final concentration of 125mM, and the cells were washed twice in cold PBS. The cell pellet was resuspended in cold sonication buffer (50mM Tris-HCl pH8, 10mM EDTA, 1\% SDS) supplemented with protease inhibitors (Complete, Roche), and chromatin was sheared to an average size of 200-500bp by sonication for 35 cycles (30s ON/30s OFF) using a Bioruptor Twin (Diagenode). After centrifugation for 10min at 20,000g, 4°C, the sonicate was pre-cleared for 2h at 4°C with 65μl of Dynabeads™ M-280 sheep anti-rabbit IgG (Life Technologies) and a 1\% input chromatin sample was set aside. The rest of the sample was diluted 10 fold in ChIP buffer (16.7mM Tris pH8 1.2mM EDTA, 167mM NaCl, 1.1\% Triton X-100) supplemented with protease inhibitors, and incubated overnight at 4°C with 5μl of either Abcam rabbit antibody: anti-HA (ab9110), anti-H3K4me3 (ab8580), anti-H3K36me3 (ab9050). Immunocomplexes were washed once with each of low salt buffer (20 mM Tris pH8, 150 mM NaCl, 1\% Triton X- 100, 0.1\% SDS, 2mM EDTA), high salt buffer (20 mM Tris pH8, 500 mM NaCl, 1\% Triton X-100, 0.1\% SDS, 2mM EDTA), LiCl buffer (10mM Tris pH8, 0.25M LiCl, 1\% NP-40, 1\% sodium deoxycholate, 1mM EDTA) and TE buffer (10mM Tris pH8, 1mM EDTA) and eluted from the beads in 100 mM NaHCO3, 1\% SDS for 30 min at 65°C with shaking. Both input and ChIP samples were reverse crosslinked overnight at 65°C in the presence of 200mM NaCl, and proteins were digested for 90 min at 45°C by addition of proteinase K (0.3ug/ul final concentration). DNA was purified using the MinElute reaction cleanup kit (QIAGEN), and quantified using the dsDNA HS assay kit and a Qubit® 2.0 Fluorometer (Invitrogen).

ChIP against DMC1 was performed from \textit{Zcwpw1\textsuperscript{-/-}} testes using the published method by \parencite{Khil2012Sensitive} with some modifications listed here. Chromatin shearing was carried out in 20mM Tris-HCl pH8, 2mM EDTA, 0.1\% SDS using a Bioruptor Pico sonicator (Diagenode) for 4 cycles of 15s ON/45s OFF. ChIP was performed in 10mM Tris-HCl pH8, 1mM EDTA, 0.1\% Sodium Deoxycholate, 1\% Triton X-100, 500mM NaCl using 5μg of mouse anti-DMC1 2H12/4 (Novus NB100-2617) pre-bound to 50μl of Dynabeads™ M-280 sheep anti-mouse IgG (Life Technologies) .

\subsubsection{Sequencing}
ZCWPW1 ChIP and input libraries from transfected cells were prepared by the Oxford Genomics Centre at the Wellcome Centre for Human Genetics (Oxford, UK) using the Nextera DNA Library Prep Kit and established Illumina protocols, and sequenced on a HiSeq 4000 platform (75bp paired end reads, 48 million reads/sample). DMC1 ChIP libraries from \textit{Zcwpw1\textsuperscript{-/-}} testes were prepared and sequenced as described previously \parencite{Davies2016Reengineering} on an Illumina HiSeq2500 platform (Rapid Run, 51bp paired end reads, 110 million reads/sample). 

\subsubsection{Read Mapping}
For the HEK293 experiments, reads were mapped to either hg38 (NCBI’s GCA 000001405.15 GRCh38 no alt's plus hs38d1 analysis set) using bwa mem (version 0.7.17-r1188) (Li, 2013). Duplicates were removed using picard’s markDuplicates (version 2.20.4-SNAPSHOT). Unmapped, mate unmapped, non primary alignment, failing platform, and low MAPQ reads were removed using samtools with parameters “-q 30 -F 3852 -f 2 -u” (version 1.9 (using htslib 1.9)). Other unmapped reads and secondary alignments were removed using samtools fixmate. Fragment position bed files were created using bedtools bamtobed (v2.28.0).

For the DMC1 mouse experimental data, we processed the data following the algorithm provided by \parencite{Khil2012Sensitive} to map the reads to the mouse mm10 reference genome \parencite{Lunter2011Stampy}, and obtain type I reads.

\subsubsection{Peak Calling}
We called DMC1 peaks, as described previously \parencite{Davies2016Reengineering}. For the HEK293 experiments, peaks for ZCWPW1, PRDM9, H3K4me3, and H3k36me3 were called using a peak caller previously described \parencite{Davies2016Reengineering} and available at \url{https://github.com/MyersGroup/PeakCaller}. Single base peaks were called with parameters pthresh $10^{-6}$ and peakminsep 250 on 22 autosomes. For each experiment, we used available IP replicates, and sequenced input DNA to estimate background, in calling peaks. Most of our analyses refer to these peaks, except that figures \ref{fig:Allele_Specific}A plots the change in ZCWPW1 occupancy following PRDM9 transfection, for either the chimp or human PRDM9 alleles. To estimate ZCWPW1 levels with vs. without PRDM9, we called ZCWPW1 peaks in HEK293T cells also transfected with PRDM9 as before, but now replacing the ``input'' lane with the IP data for ZCWPW1 in cells not transfected with PRDM9. Figure \ref{fig:ROC} also uses this measure, of ZCWPW1 recruitment attributable to PRDM9 binding, to predict DMC1 levels, while \ref{fig:proportions} (right panel) checks that changes in ZCWPW1 occupancy occur largely at PRDM9-bound sites. We analysed previously published H3K4me3 and SPO11 read data \parencite{Davies2016Reengineering, Lange2016Landscape}.

\subsubsection{Enrichment Profiles for HEK293T experiments}
Peaks were filtered if: center within 2.5kb of PRDM9 independent H3K4me3 (promoters), input coverage <=5, in top 5 by likelihood, greater than 99.9th percentile input coverage. For human PRDM9 peaks the PRDM9 motif position \parencite{Altemose2017map} was inferred using the getmotifs function in MotifFinder within the 300bp region around the peak (with parameters alpha=0.2, maxits=10, seed=42, stranded\_prior=T), and peaks were recentered and stranded at these locations. Peaks within 4kb of one another are removed to avoid double counting. Mean coverage was calculated with bwtool v1.0 using the aggregate command with parameters “-fill=0 -firstbase” at a width of $\pm2$kb. Each profile was normalised by the total coverage of all of the fragments as well as by the input profile. Random profiles were created using bedtools random with 10,000 locations, with seed 72346, and were also filtered such that non were within 4kb of one another.

\subsubsection{DSB Profiles}
Mapped type 1 reads were filtered by removing non-canonical chromosomes (with underscores in names), sex chromosomes, and mitochondrial reads. Bedgraphs were converted to bigWig format using UCSC bedGraphToBigWig v4. Profiles were created using bwtool v1.0 using the aggregate command with parameters 5000:2000 and -fill=0. B6 wild type hotspot locations were filtered to remove non-autosomal chromosomes, retain only B6 allele hotspots, and remove hotspots with PRDM9-independent H3K4me3. \textit{Prdm9\textsuperscript{-/-}} hotspots were filtered to remove X chromosome. Motif centered coordinates were used for wild type hotspots. To normalise background signal, the mean signal between -5000 and -3000 was subtracted from each strand-mouse combination. Mean coverage was normalised by the sum of coverage over each strand-mouse combination across both WT and KO locations.

\subsubsection{Mapping of Alu CpGs}
Alu locations were downloaded from UCSC tables and filtered for Alu repeats with a width between 250 and 350bp. DNA sequence was extracted from the genome (Ensembl 95 h38 primary assembly) at these locations using the bedtools getfasta command and CpG dinucleotides were counted using ‘stringr::str\_locate\_all’ command in R.

\subsubsection{CpG Methylation}
\label{sec:cpgmeth}
Data is from \parencite{Libertini2015Overexpression} (GSE51867), non CpG methylated sites were removed, and the two strands were summed. High copy number (>1.5) CpGs were removed.

\subsubsection{DMC1 Prediction using ZCWPW1 and PRDM9 binding strength}
\label{sec:prediction}
PRDM9-dependent ZCWPW1 was force-called at the human PRDM9 peaks. DMC1 sites are from \parencite{Pratto2014DNA} (GSE59836) and were subsetted to ‘AorB’ autosomal peaks with a width less than 3kb. These peaks were then trimmed to a maximum of 800bp before overlapping with PRDM9 peaks to create a binary target variable. Regions with input coverage <=5 in either predictor were removed in addition to outliers with input > 200 and or Zcwpw1 enrichment >10. Chromosomes 1, 3, and 5 were used as the test data with the remaining autosomes being the training data.

\subsubsection{Heatmaps}
\label{sec:heatmaps}
Regions were extracted from bigWig using bwtool matrix -fill=0 -decimals=1 -tiled-averages=5, and width parameters 2000. Coverage was normalised by total coverage (scaled by $10^{10}$). A pseudocount of 1 was added to both input and sample, and the sample was then normalised by the input for each region. Values outside the quantile range 0.01-0.99 were thresholded. The profile plots are created by taking the ratio of the mean coverage of the sample and input separately. Ordering of the regions was determined by the mean coverage of a 200bp window centered on the peak center.

\subsubsection{Software}
Computational analysis was performed using R \parencite{RCoreTeam2018Language}, and Snakemake \parencite{Koster2012Snakemake}. BigWig aggregations by bwtool \parencite{Pohl2014bwtool}, BED manipulation with BEDtools \parencite{Quinlan2010BEDTools}, and FASTA processing with seqkit \parencite{Shen2016SeqKit}. We also used BigWig tools \parencite{Kent2010BigWig}, IGV \parencite{Thorvaldsdottir2013Integrative}, and GNU parallel \parencite{Tange2011GNU}. Plots were created using the ggplot2 package \parencite{Wickham2016ggplot2} and extensions ggforce \parencite{Pedersen2016ggforce}, RColorBrewer \parencite{Neuwirth2014RColorBrewer}, viridis \parencite{Garnier2018viridis}, drawProteins \parencite{Brennan2018drawProteins}, and cowplot \parencite{Wilke2018cowplot}. In addition the following R packages were used: data.table \parencite{Dowle2019data}, mgcv \parencite{Wood2011Fast}, PRROC \parencite{Grau2015PRROC}, sangerseqR \parencite{Hill2014Poly}, and ComplexHeatmap \parencite{Gu2016Complex}.

\subsection{Data and Code availability}
Code is available at \url{www.github.com/MyersGroup/Zcwpw1} archived at Zenodo (DOI: \href{http://www.doi.org/10.5281/zenodo.3559758}{10.5281/zenodo.3559758}). Raw and processed data from ZCWPW1 and DMC1 ChIPseq is available at GEO accession GSE141516 (access token \url{urehiygebxqltaj}).

DMC1 SSDS ChIPseq from human testis is from \parencite{Pratto2014DNA} (GSE59836). H3K4me3 ChIPseq from mice testis is from \parencite{Davies2016Reengineering} (GSE73833). SPO11 ChIPseq from mice testis is from \parencite{Lange2016Landscape} (GSE84689). DMC1 SSDS ChIPseq from WT B6 mouse testis is from \parencite{Brick2012Genetic} (GSE35498). PRDM9, H3K4me3 and H3K36me3 ChIPseq from HEK293 cells is from \parencite{Altemose2017map} (GSE99407). Whole-genome bisulfite sequencing (BS-seq) of HEK293 cells is from \parencite{Libertini2015Overexpression} (GSE51867).

