\begin{savequote}[8cm]
\textit{S'il se produit la quatre spores et pas deux, c'est qu'elles doivent avoir chacune quelque chose de particulier.}

If four spores occur and not two, then each must have something special.
\qauthor{--- \cite{Janssens1909theorie}}
\end{savequote}

\chapter{\label{ch:4-discuss}Perspectives}

\minitoc


\section{Other Single Cell RNAseq studies}
Whilst there were no single cell RNAseq studies of spermatogenesis when this research was initiated, it was clearly a popular idea as there are now an increasing number of such studies (22 as of December 2019), briefly reviewed and compared with our results below. Of note, only two other studies examine mutant phenotypes, and one other study examines transcription factor binding: we discuss these further below. Finally no other study has used and SDA based approach.

%pre-meiotic foetal development~\cite{Li2017SingleCell}.

\begin{table}[]
	\begin{tabular}{@{}llll@{}}
		\toprule
		Study                                                & Cells   & Organism       & Method \\ \midrule
		\cite{Hermann2018Mammalian}		& 62,141  & Human \& Mouse & Chromium \& SMARTer (C1) \\
		\cite{Ernst2019Staged}					& 53,510  & Mouse          & Chromium\\
		\cite{Green2018Comprehensive}	& 34,633  & Mouse          & Drop-seq    \\
		\cite{Sohni2019Neonatal}				& 33,585  & Human          & Chromium             \\
		\cite{Jung2019Unified}              & 20,322  & Mouse          & DropSeq                  \\
		\cite{Han2018Mapping}               & 19,659  & Mouse          & Microwell-seq          \\
		\cite{Grive2019Dynamic}             & 15,882  & Mouse          & Chromium                \\
		\cite{Law2019Developmental}         & 10,140  & Mouse          & Chromium             \\
		\cite{La2018Identification}         & 9,424   & Mouse          & Chromium                   \\
		\cite{Fang2019Proteomics}           & 6,804   & Mouse          & Chromium                \\
		\cite{Guo2018adult}                 & 6,490   & Human          & Chromium                  \\
		\cite{Xia2019Widespread}            & 4,147   & Human \& Mouse & inDrop            \\
		\cite{Wang2018SingleCell}           & 3,028   & Human          & Smart-seq2          \\
		\cite{Lukassen2018Characterization} & 2,550   & Mouse          & Chromium        \\
		\cite{Vertesy2019Dynamics}          & 1,274   & Mouse          & SORT-seq             \\
		\cite{Chen2018Singlecell}           & 1,174   & Mouse          & Smart-seq2              \\
		\cite{Stevant2018Deciphering}       & 400     & Mouse          & SMARTer (C1)     \\
		\cite{Song2016Homeobox}             & 201     & Mouse          & SMARTer (C1)      \\
		\cite{Makino2019Single}             & 175     & Mouse          & SMARTer (C1)             \\
		\cite{Neuhaus2017Singlecell}        & 105     & Human          & Tang                    \\
		\cite{Guo2017Chromatin}             & 92      & Human          & SMARTer (C1)      \\
		\cite{Liao2019Revealing}            & 71      & Mouse          & SMARTer (C1)           \\
		Total                                                & 285,807 &                &                             \\ \bottomrule
	\end{tabular}
\end{table}

Building on a previous study \parencite{Guo2017Chromatin} of scRNAseq of 92 cells from human SSEA4+ hSSCs and c-KIT+ spermatogonia, \cite{Guo2018adult} generated a dataset of 6,490 cells using the 10X Chromium platform from 3 human donors. This study performed RNA velocity analysis on early germ cells, which revealed two sub-populations of the earliest stem cells (one steady, one committing). Interestingly, it also revealed later stem cells whose velocity vectors were pointing ``backwards'' towards less differentiated state \parencite{Guo2018adult}, consistent with previous reports of plasticity in spermatogonia \parencite{Brawley2004Regeneration, Nakagawa2010Functional, Hara2014Mouse}.

They highlight that \textit{Csf1r} was reported to be expressed only in spermatogonia in mice, but in their human data it is specifically expressed in macrophages \parencite{Guo2018adult}. However, this is not what the original study claimed, rather that there exists a very rare population of THY1+ spermatogonial stem cells that also express low levels of \textit{Csf1r}, in addition to the macrophage expression \parencite{Oatley2009Colony}. Indeed, our mouse data shows specific \textit{Csf1r} expression in macrophages, and in our dataset very few cells have detectable Thy1. They also highlight that in their human data \textit{Cxcl12} is detected in Leydig cells, but was previously reported as expressed in Sertoli cells in mice \parencite{Yang2013CXCL12}. However, the cited study only claims that CXCL12 is expressed in Sertoli cells \emph{within} the adult tubule, and it also shows that CXCL12 is detected outside the tubules in Leydig cells. In addition the expression in Sertoli cells was weakly co-localised, confined to the basement membrane, and shown using a poor marker of Sertoli cells (GATA4).

\cite{Stevant2018Deciphering}, isolated a total of 400 NR5A1-GFP+ cells by FACS from E10.5, E11.5, E12.5, E13.5, and E16.5 testis, revealing that both Sertoli and Leydig cells originate from a single common progenitor population, with Sertoli cell differentiation potentially driven by a pulse of expression of \textit{Sry}, \textit{Kdm3a}, and \textit{Nrob1} among others.

\cite{Chen2018Singlecell}, generated a dataset of 1,136 cells over 20 time points from mice with Vasa-dTomato and Lin28-YFP which were treated with retinoic acid to synchronise spermatogenesis, in addition to 38 cells from \textit{Spo11\textsuperscript{-/-}} mice. In agreement with our results they found enrichment of \textit{Prdm9}, \textit{Spo11}, \textit{Gm960}, \textit{Meiob}, \textit{Dmc1}, and \textit{Mcm8} in their leptotene \& zygotene cluster C3. They also detected novel enriched genes in this cluster \textit{Fbxo47}, \textit{Pparg}, and \textit{Ccnb3}. \textit{Fbxo47\textsuperscript{-/-}} mice were completely infertile, similar to the previously studied \textit{C. elegans} homologue \textit{prom-1} \parencite{Jantsch2007Caenorhabditis}. A follow up study found \textit{Fbxo47} to be required for DSB repair and synapsis at telomeres, potentially due to inhibition of TFR2 ubiquitination \parencite{Hua2019FBXO47}. To my knowledge, this represents the only other meiotic study (apart from our own) so far where a gene discovery from single cell data resulted in a new publication.

\cite{Chen2018Singlecell}, also identified Sox30 as highly expressed in pachytene and generated \textit{Sox30\textsuperscript{-/-}} mice, which were infertile, in agreement with an earlier study \parencite{Feng2017SOX30}, and SOX30 ChIPseq detected some peaks in promoter regions for genes that were differentially regulated in \textit{Sox30\textsuperscript{-/-}} mice. \cite{Chen2018Singlecell}, were also able to profile alternative splicing events, enabling characterisation of \textit{Spo11} isoform expression ($\beta$ in leptotene and $\alpha$ in pachytene). They also investigated MSCI and identified 150 genes as ``escaping'' MSCI (defined by them as expression in diplotene stage). However, none of these genes show convincing evidence of MSCI escape in our dataset. \cite{Vertesy2019Dynamics}, also find genes escaping MSCI in a dataset of 1,274 cells from Dazl-GFP mice (up to the start of pachytene, median transcript count 20,488/cell), particularly \textit{Slitrk2}. However, this gene was below the detection threshold in our dataset.

\cite{Green2018Comprehensive} used DropSeq to generate a dataset of 34,633 cells from mice testis (average 6,205 UMIs/cell). They were able to identify two new somatic populations which they describe as an innate lymphoid type II immune cell, and a mesenchymal cell. Whilst it's unclear if our dataset also contains the immune cell based on the marker genes they identified, it most certainly contains the mesenchymal cell which they validated using a Tcf21-creERT2; tdTomato mouse. We identified these cells as telocytes based on previous work defining the markers of these cells \parencite{Marini2018Reappraising}. By using Sox9-EGFP and Amh-cre;mTmG transgenic lines \cite{Green2018Comprehensive} were able to enrich for Sertoli cells, resulting in 9 sub-clusters. Interestingly they also found Prm2 as a marker of some Sertoli cells, and with intronic/UTR probes were able to determine that these transcripts were transcribed in round spermatids but persist in Sertoli cells after phagocytosis.

\cite{Green2018Comprehensive} is the only other study to perform transcription factor motif analysis in addition to our own, and they reach some different conclusions worth discussing. Compared to our motif analysis, one major difference is that they are using MEME-ChIP (which is a combination of MEME, DREME [a non probabilistic regular expression based motif finder], and CentriMo [which looks for central enrichment of known motifs]). The CentriMo manual cautions that when the number of sequences is large, motifs that are only slightly similar can show significant enrichment. It appears that many of their \emph{unknown} motifs are from the DREME method.  Our most commonly found motif is the Sp1 family motif, and while they don't find this they do find a reverse complemented version which they assign to Bcl6b. However, the Bcl6b motif in the HOCOMOCO database does not appear greatly similar to the motif they find. Some motifs which they classify as unknown clearly match motifs we found with known transcription factors such as ZNF143 in their gene group 1. They also find a motif which matches well to our CREM-t motif in their group 6 but they do not link it to CREM, likely due to the short motifs that DREME typically produces. Many of their other unknown motifs are however very long with limited information score diversity. In fact they are close to the maximum length possible for DREME. It is plausible these ``motifs'' are actually just the promoter sequence of ampliconic genes such as the tasukan family of genes on chromosome 14 and so their enrichment does not imply specific transcription factor binding.

\cite{Grive2019Dynamic} sampled a total of 15,882 cells form five different postnatal timepoints during the initial wave of spermatogenesis (PND 6, 14, 18, 25, and 30) and so were able to profile how gene expression changes during this testis maturation process as well as asses the proportions of different cells types at each stage. For example DNA repair genes such as \textit{FancJ} (\textit{Brip1}), \textit{Brca1}, \textit{Rad51}, and \textit{Atm} apparently have increased expression in adult testis compared to the equivalent stage during the first wave of spermatogenesis.

\cite{Ernst2019Staged}, generated a dataset of 53,510 single cells from both adult (8-9 weeks) and juvenile (PND 5â€“35, in 5 day steps) mice. These results agree with our own including for example the identification of \textit{Pou5f2} as a marker of late prophase I. In addition, they profiled mice with an additional chromosome - human chromosome 21 (Tc1 mice) - but detected minimal differences in transcription other than relative lack of post-meiotic cell types due to metaphase arrest, reminiscent of our \textit{Mlh3} mutants. \cite{Ernst2019Staged} also looked at MSCI dynamics and in agreement with our results also found a sharp drop in sex to autosome expression ratio at the zygotene-pachytene transition followed by a gradual reactivation. They also highlighted the Ssxb family as one of the first to be expressed post MSCI, and confirmed expression using ISH. This study also performed CUT\&RUN to assay H3K4me3, H3K9me3, and K3K27ac, confirming an enrichment of H3K9me3 on the X chromosome of spermatids, and at spermatid specific genes in meiosis, likely deposited by SETDB1 \parencite{Hirota2018SETDB1}.

\cite{Xia2019Widespread} generated a dataset of 4,147 cells from both human and mouse testis (average UMI 7,459) and note that >90\% of all protein coding genes are expressed in germ cells (c.f. 62\% in somatic cells), in line with previous observations \parencite{Soumillon2013Cellular, Schmidt1996Transcriptional}. They propose a model of pervasive transcriptional scanning in the testis germ cells in order to promote transcription coupled repair and hence reduced germline mutations. Consistently they found reduced mutation rates in expressed vs unexpressed genes within the testis and in addition this effect was higher on the template strand (and on the coding strand upstream). They also found human cells had much higher expression of some genes, such as \textit{CXCL6} and \textit{GAPDH}, whereas mouse had higher expression of \textit{Fabp9} and \textit{Sord} \parencite{Xia2019Widespread}.

\cite{Law2019Developmental}, generated a dataset of 10,140 cells from E16.5, P0, P3, and P6 ID4-eGFP mice (median 20,546 UMI/cell), and showed that ID4-eGFP+ prospermatogonia from E16.5 mice were able to establish colonies in adult germ cell depleted recipient testis.

\cite{Fang2019Proteomics}, generated a dataset of 3,659 wild type and 3,145 \textit{Akap4\textsuperscript{-/-}} cells, a major component of the fibrous sheath of spermatids \parencite{Eddy2003Fibrous}. With our own, this is the only other study, in addition to the human chromosome 21 knock in by \cite{Ernst2019Staged}, that performed single cell RNAseq on mutant mice.

\cite{Lukassen2018Characterization}, generated a dataset of 2,550 cells from adult mice. They claim high expression of \textit{Pou5f1} in round spermatids but no expression was seen at this stage in our study. They also detect expression of \textit{Kit} in round spermatids, and indeed in our study we detect the occasional transcript at this stage. 

%\cite{Hermann2018Mammalian}, generated the largest dataset of 62,141 cells from both mice and humans. As part of a multi tissue whole organism atlas (>400,000 cells from >50 mouse tissues and cell cultures) \cite{Han2018Mapping} generated a dataset of 19,659 cells from testis using Microwell-Seq.

\cite{Wang2018SingleCell}, sequenced 2,854 cells from normal humans, and 174 from a nonobstructive azoospermia patient. Many of the marker genes highlighted and validated by ISH agree with our assignments in mice including \textit{Hmga1} in spermatogonia, \textit{Ovol2} in pachytene and diplotene, and \textit{Tex29} in early spermatids. \cite{Sohni2019Neonatal}, generated a dataset of 18,723 and 14,862 cells from human adult and newborn testis respectively and so were able to compare and investigate neonatal germ cell development.

In comparison to other datasets our dataset has the lowest UMI count per cell (1,312) - 1/5th of the other DropSeq dataset, and 1/10th of the 10X Chromium datasets. Despite this, by using matrix factorisation rather than hard clustering as in the other studies, it appears we are able to gain as many and often more biological insights from our data.

\section{Observational Extensions}
% ATAC, proteome, chipseq - single cell, prior info (motifs), lineage tracing?, spatial seq, 
% Other potential target genes

In total these studies represent over 275,000 cells and so there may be significant advantages in combining them. One challenge of this would be dealing with batch effects from different species, genetic backgrounds, developmental time points, cell capture technology, and sequencing platforms. We have shown SDA can capture batch effects and so could provide one possible solution to this problem, although we have not tested for example combining 10X and DropSeq or human and mouse. Even within one of these studies there could be significant advantages to using a matrix factorisation approach. For example with multiple developmental time points you might expect to find a main component for each cell type, and then ``modifier'' components representing how the transcription of that cell type changes before, during and after the first wave of spermatogenesis.

Whilst some studies \parencite{Ernst2019Staged} have used other (bulk) assays alongside scRNAseq, there is much to be gained from single cell resolution profiling of protein abundance, protein-DNA binding, chromatin accessibility, methylation/DNA modifications, nuclear structure, and spatial aspects; even more so when these modes are profiled simultaneously in single cells. The main challenge for this has been experimental limitations but now many methods are available to profile one or multiple of these aspects \parencite[reviewed in][]{Chappell2018SingleCell, Hu2018Single, Stuart2019Integrative, Heriche2019Integrating}. SDA is also able to perform group factor analysis \parencite{Hore2015Latent}, and a similar analysis package has been applied to single cell multiomic data \parencite{Argelaguet2019Multiomics}.

Chromatin accessibility assays would be particularly interesting given the dramatic chromatin remodelling that occurs during spermatogenesis. The genome also undergoes dramatic de-methylation and re-methylation during gametogenesis and so methylation assays would also be intriguing. Protein-DNA assays such as single cell ChIP-seq could help to directly elucidate the gene regulatory programme in much more detail than we have been able to achieve here by inferring motifs. Due to the spatial layout of the different stages of spermatogenesis in addition to the cycle of the seminiferous epithelium, spatial sequencing seems particularly apt for the investigation of transcriptome of the testis. The spatial structure would help to more precisely state the cell identity of ambiguous transcriptomes, and would help to reveal additional structure, for example how the supporting Sertoli cells differ at different stages of the seminiferous cycle. Many single cell (and even bulk) studies of the testis have focused on the spermatogonia in an effort to identify the true spermatogonial stem cell. Single cell genetic lineage tracing could help to answer this question by providing a molecular readout of the cell division history for each cell \parencite[reviewed in][]{Baron2019Unravelling, McKenna2019Recording}. For many of these aspects there is already some information known for example which motifs are present at which promoters, and which genes cause infertility or are involved in meiotic processes. Incorporating this prior and multi-omic information into the decomposition would likely help to disambiguate co-expressed gene sets into finer more distinct functional groupings, in addition to providing insight into the molecular processes that underlie the functional changes that occur during the hugely complex and intricate performance that is spermatogenesis.

\section{Other Zcwpw1 Papers \& Extensions}

Prior to our study the only work linking \textit{Zcwpw1} to meiosis was the study by \cite{Soh2015Gene}, who identified that \textit{Zcwpw1} was specifically expressed in foetal prophase I in female mice. This lead to the work by \cite{Li2019histone} revealing that \textit{Zcwpw1} is required for fertility in males, and this work was compared to our results in chapter \ref{ch:3-Zcw}. Since finishing our study, two other reports have been made public investigating the role of \textit{Zcwpw1} in meiotic recombination.

One of these studies, by \cite{Mahgoub2019Dual}, was motivated by a reanalysis of the data from \cite{Chen2018Singlecell}, showing high co-expression of \textit{Zcwpw1} with \textit{Prdm9}. They also show co-evolution of \textit{Zcwpw1} with \textit{Prdm9}, but our work specifically highlights the association with the SET and SSXRD domains of \textit{Prdm9} (required for methylation) compared to the other domains.

Both studies used the same \textit{Zcwpw1\textsuperscript{-/-}} mouse, and both studies found DSB positioning is unchanged. They differ in that Mahgoub and colleges used ENDseq, showing altered DSB repair post homologue invasion, while we used DMC1 SSDS showing altered repair timing at individual hotspots and its association with PRDM9 binding.

The two studies also investigate different model organisms. We used an \textit{in vitro} system to study the human ZCWPW1 and PRDM9 proteins, identifying >800,000 ZCWPW1 binding sites ($p<10^{-6}$), the top \textasciitilde10,000 of which are almost all PRDM9-bound sites. Mahgoub and colleagues used mice, identifying 4,300 ZCWPW1 binding sites ($p<10^{-3}$), again mainly PRDM9-bound sites. The additional weaker peaks we found revealed a more subtle CpG influence on ZCWPW1 binding, which also impacts stronger peaks. While we used chimp PRDM9 to show allele specificity, Mahgoub and colleagues used hybrid mice.

Some analyses were unique to each study. For example we counted stage specific DMC1 by immunofluorescence, revealing DSB repair delay and identifying which chromosomes fail to synapse in \textit{Zcwpw1\textsuperscript{-/-}} mice. We also investigated the subnuclear positioning of ZCWPW1 using immunofluorescence, discovering an interesting new telomeric localisation. Using biotin-streptavidin pulldown assays, Mahgoub and colleagues show that \emph{dual} modified H3K4me3-H3K36me3 peptides had the highest binding affinity for ZCWPW1. While we showed that PRDM9 bound sites (with the dual mark) are stronger recruiters of ZCWPW1 than H3K4me3 alone.

Another group, having previously generated a different \textit{Zcwpw1\textsuperscript{-/-}} mouse \parencite{Li2019histone}, generated a new mutant with three point mutations in the zf-CW domain: \textit{Zcwpw1\textsuperscript{W247I/E292R/W294P}} rendering the H3K4me3 recognition capacity non-functional \parencite{Huang2019histone}. This mouse was also infertile with complete testicular azoospermia and incomplete synapsis observed by SYCP1/3 staining. ChIPseq against ZCWPW1 revealed expected high overlap with H3K4me3 and DMC1 marks in WT testis and lack of peaks in \textit{Zcwpw1\textsuperscript{-/-}} and \textit{Zcwpw1\textsuperscript{W247I/E292R/W294P}} mice. ChIPseq against ZCWPW1 in \textit{Prdm9\textsuperscript{-/-}} mice resulted in very few peaks of which >80\% were within 5kb of a TSS.

While these papers clearly agree that ZCWPW1 aids DSB repair, exactly how this is achieved remains unclear. In addition, the missing link between PRDM9 and SPO11 recruitment remains missing. As previously discussed \textit{Zcwpw2} is a promising candidate for this role, just as \textit{Zcwpw1} was, and further work will prove what (if any) involvement \textit{Zcwpw2} has in meiotic recombination.

It is likely that many other genes revealed by single cell RNA sequencing will be key players in meiosis and spermatogenesis and hopefully this work will enable some of the many mysteries to be unveiled.

